<?xml version='1.0' encoding='utf-8'?>
<testsuites tests="147" failures="29" errors="0" skipped="45" time="0.0"><testsuite name="SoftwareManagement.Plugin.apama_plugin_install" tests="3" errors="0" failures="0" skipped="1" time="0"><testcase classname="ApamaPluginInstallTest" name="SoftwareManagement.Plugin.apama_plugin_install">
		<skipped message="SKIPPED: Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests" />
		<system-out>10:24:41 INFO  ==============================================================
10:24:41 INFO  Id:    SoftwareManagement.Plugin.apama_plugin_install
10:24:41 INFO  Dir:   plugin_apama/
10:24:41 INFO  Title: Validate apama plugin install
10:24:41 INFO  ==============================================================
10:24:41 INFO  Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests ... skipped
10:24:41 WARN  Aborted test due to SKIPPED outcome
10:24:41 INFO  
10:24:41 INFO  Test duration: 0.05 secs
10:24:41 INFO  Test final outcome:  SKIPPED
10:24:41 INFO  Test outcome reason: Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests
10:24:41 INFO  
</system-out>
	</testcase>
<testcase classname="ApamaPluginInstallTest" name="SoftwareManagement.Plugin.apama_plugin_install" />
<testcase classname="ApamaPluginInstallTest" name="SoftwareManagement.Plugin.apama_plugin_install" />
</testsuite><testsuite name="SoftwareManagement.Plugin.apama_plugin_mon_support" tests="3" errors="0" failures="0" skipped="1" time="0"><testcase classname="ApamaPluginMonSupportTest" name="SoftwareManagement.Plugin.apama_plugin_mon_support">
		<skipped message="SKIPPED: Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests" />
		<system-out>10:24:41 INFO  ==============================================================
10:24:41 INFO  Id:    SoftwareManagement.Plugin.apama_plugin_mon_support
10:24:41 INFO  Dir:   plugin_apama/
10:24:41 INFO  Title: Validate mon file support of apama plugin
10:24:41 INFO  ==============================================================
10:24:41 INFO  Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests ... skipped
10:24:41 WARN  Aborted test due to SKIPPED outcome
10:24:41 INFO  
10:24:41 INFO  Test duration: 0.03 secs
10:24:41 INFO  Test final outcome:  SKIPPED
10:24:41 INFO  Test outcome reason: Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests
10:24:41 INFO  
</system-out>
	</testcase>
<testcase classname="ApamaPluginMonSupportTest" name="SoftwareManagement.Plugin.apama_plugin_mon_support" />
<testcase classname="ApamaPluginMonSupportTest" name="SoftwareManagement.Plugin.apama_plugin_mon_support" />
</testsuite><testsuite name="SoftwareManagement.Plugin.apama_plugin_remove" tests="3" errors="0" failures="0" skipped="1" time="0"><testcase classname="ApamaPluginRemoveTest" name="SoftwareManagement.Plugin.apama_plugin_remove">
		<skipped message="SKIPPED: Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests" />
		<system-out>10:24:41 INFO  ==============================================================
10:24:41 INFO  Id:    SoftwareManagement.Plugin.apama_plugin_remove
10:24:41 INFO  Dir:   plugin_apama/
10:24:41 INFO  Title: Validate apama plugin remove
10:24:41 INFO  ==============================================================
10:24:41 INFO  Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests ... skipped
10:24:41 WARN  Aborted test due to SKIPPED outcome
10:24:41 INFO  
10:24:41 INFO  Test duration: 0.05 secs
10:24:41 INFO  Test final outcome:  SKIPPED
10:24:41 INFO  Test outcome reason: Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests
10:24:41 INFO  
</system-out>
	</testcase>
<testcase classname="ApamaPluginRemoveTest" name="SoftwareManagement.Plugin.apama_plugin_remove" />
<testcase classname="ApamaPluginRemoveTest" name="SoftwareManagement.Plugin.apama_plugin_remove" />
</testsuite><testsuite name="SoftwareManagement.Plugin.apama_plugin_update" tests="3" errors="0" failures="2" skipped="1" time="0"><testcase classname="ApamaPluginUpdateTest" name="SoftwareManagement.Plugin.apama_plugin_update">
		<skipped message="SKIPPED: Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests" />
		<system-out>10:24:41 INFO  ==============================================================
10:24:41 INFO  Id:    SoftwareManagement.Plugin.apama_plugin_update
10:24:41 INFO  Dir:   plugin_apama/
10:24:41 INFO  Title: Validate apama plugin updating one project with another
10:24:41 INFO  ==============================================================
10:24:42 INFO  Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests ... skipped
10:24:42 WARN  Aborted test due to SKIPPED outcome
10:24:42 INFO  
10:24:42 INFO  Test duration: 0.02 secs
10:24:42 INFO  Test final outcome:  SKIPPED
10:24:42 INFO  Test outcome reason: Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests
10:24:42 INFO  
</system-out>
	</testcase>
<testcase classname="ApamaPluginUpdateTest" name="SoftwareManagement.Plugin.apama_plugin_update">
		<failure message="BLOCKED: sudo&lt;plugin_install&gt; returned non-zero exit code 2" type="BLOCKED" />
		<system-out>11:17:08 INFO  ==============================================================
11:17:08 INFO  Id:    SoftwareManagement.Plugin.apama_plugin_update
11:17:08 INFO  Dir:   plugin_apama/
11:17:08 INFO  Title: Validate apama plugin updating one project with another
11:17:08 INFO  ==============================================================
11:17:08 INFO  Assertion that path exists=False for "/etc/tedge/apama/project" ... passed
11:17:08 INFO  Executed sudo&lt;create_project_dir&gt;, exit status 0
11:17:09 INFO  Executed sudo&lt;unzip_apama_project&gt;, exit status 0
11:17:09 INFO  Executed sudo&lt;move_apama_project&gt;, exit status 0
11:17:10 INFO  Executed sudo&lt;restart_apama_service&gt;, exit status 0
11:17:11 INFO  Executed sudo&lt;apama_correlator_wait&gt;, exit status 0
11:17:11 INFO  Assertion that path exists=True for "/etc/tedge/apama/project" ... passed
11:17:11 INFO  Executed sudo&lt;apama_list_monitors&gt;, exit status 0
11:17:11 INFO  Grep on file /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_apama/apama_plugin_update/Output/linux/apama_list_monitors.out ... passed
11:17:12 WARN  Executed sudo&lt;plugin_install&gt;, exit status 2
11:17:12 INFO  Contents of plugin_install.err: 
  ERROR: Directory not empty (os error 39)
  -----

11:17:12 WARN  sudo&lt;plugin_install&gt; returned non-zero exit code 2 ... blocked [run.py:35]
11:17:12 WARN  Aborted test due to BLOCKED outcome
11:17:12 INFO  
11:17:12 INFO  cleanup:
11:17:12 INFO  Removing apama project
11:17:28 INFO  Executed sudo&lt;stop_apama_service&gt;, exit status 0, duration 15 secs
11:17:29 INFO  Executed sudo&lt;remove_apama_project&gt;, exit status 0
11:17:29 INFO  Executed sudo&lt;remove_apama_tmp_dir&gt;, exit status 0
11:17:29 INFO  
11:17:29 INFO  Test duration: 21.68 secs
11:17:29 INFO  Test final outcome:  BLOCKED
11:17:29 INFO  Test outcome reason: sudo&lt;plugin_install&gt; returned non-zero exit code 2
11:17:29 INFO  
</system-out>
	</testcase>
<testcase classname="ApamaPluginUpdateTest" name="SoftwareManagement.Plugin.apama_plugin_update">
		<failure message="BLOCKED: sudo&lt;plugin_install&gt; returned non-zero exit code 2" type="BLOCKED" />
		<system-out>11:23:52 INFO  ==============================================================
11:23:52 INFO  Id:    SoftwareManagement.Plugin.apama_plugin_update
11:23:52 INFO  Dir:   plugin_apama/
11:23:52 INFO  Title: Validate apama plugin updating one project with another
11:23:52 INFO  ==============================================================
11:23:52 INFO  Assertion that path exists=False for "/etc/tedge/apama/project" ... passed
11:23:52 INFO  Executed sudo&lt;create_project_dir&gt;, exit status 0
11:23:53 INFO  Executed sudo&lt;unzip_apama_project&gt;, exit status 0
11:23:53 INFO  Executed sudo&lt;move_apama_project&gt;, exit status 0
11:23:54 INFO  Executed sudo&lt;restart_apama_service&gt;, exit status 0
11:23:55 INFO  Executed sudo&lt;apama_correlator_wait&gt;, exit status 0
11:23:55 INFO  Assertion that path exists=True for "/etc/tedge/apama/project" ... passed
11:23:56 INFO  Executed sudo&lt;apama_list_monitors&gt;, exit status 0
11:23:56 INFO  Grep on file /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_apama/apama_plugin_update/Output/linux/apama_list_monitors.out ... passed
11:23:56 WARN  Executed sudo&lt;plugin_install&gt;, exit status 2
11:23:56 INFO  Contents of plugin_install.err: 
  ERROR: Directory not empty (os error 39)
  -----

11:23:57 WARN  sudo&lt;plugin_install&gt; returned non-zero exit code 2 ... blocked [run.py:35]
11:23:57 WARN  Aborted test due to BLOCKED outcome
11:23:57 INFO  
11:23:57 INFO  cleanup:
11:23:57 INFO  Removing apama project
11:24:12 INFO  Executed sudo&lt;stop_apama_service&gt;, exit status 0, duration 15 secs
11:24:14 INFO  Executed sudo&lt;remove_apama_project&gt;, exit status 0
11:24:15 INFO  Executed sudo&lt;remove_apama_tmp_dir&gt;, exit status 0
11:24:15 INFO  
11:24:15 INFO  Test duration: 23.15 secs
11:24:15 INFO  Test final outcome:  BLOCKED
11:24:15 INFO  Test outcome reason: sudo&lt;plugin_install&gt; returned non-zero exit code 2
11:24:15 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="SoftwareManagement.sm_apt_install_download_path" tests="3" errors="0" failures="2" skipped="1" time="0"><testcase classname="AptInstallWithDownloadPath" name="SoftwareManagement.sm_apt_install_download_path">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it" />
		<system-out>10:34:23 INFO  ==============================================================
10:34:23 INFO  Id:    SoftwareManagement.sm_apt_install_download_path
10:34:23 INFO  Dir:   software_management_end_to_end/
10:34:23 INFO  Title: Install package to directory specified in tedge config set tmp.path
10:34:23 INFO  ==============================================================
10:34:23 INFO  Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it ... skipped
10:34:23 WARN  Aborted test due to SKIPPED outcome
10:34:23 INFO  
10:34:23 INFO  Test duration: 0.06 secs
10:34:23 INFO  Test final outcome:  SKIPPED
10:34:23 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it
10:34:23 INFO  
</system-out>
	</testcase>
<testcase classname="AptInstallWithDownloadPath" name="SoftwareManagement.sm_apt_install_download_path">
		<failure message="BLOCKED: AssertionError:" type="BLOCKED" />
		<system-out>10:58:47 INFO  ==============================================================
10:58:47 INFO  Id:    SoftwareManagement.sm_apt_install_download_path
10:58:47 INFO  Dir:   software_management_end_to_end/
10:58:47 INFO  Title: Install package to directory specified in tedge config set tmp.path
10:58:47 INFO  ==============================================================
10:58:47 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
10:58:47 INFO  EnvironmentC8y Setup
10:58:48 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
10:59:02 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
10:59:04 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
10:59:05 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
10:59:06 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
10:59:08 ERROR No operations found, assuming it passed
10:59:08 INFO  Assert that (False == value) with value=False ... passed
10:59:09 INFO  Executed sudo, exit status 0
10:59:09 INFO  Executed sudo, exit status 0
10:59:10 INFO  Executed sudo, exit status 0
10:59:14 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
10:59:14 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 3 seconds
10:59:14 INFO  Delaying execution by 8 seconds
10:59:35 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 13 secs
10:59:36 INFO  Response status: 201
10:59:36 INFO  Response to action: {
    "creationTime": "2022-05-02T10:59:36.182Z",
    "deviceName": "m32sd10c",
    "deviceId": "18127529",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18152682",
    "id": "18152682",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'name': 'rolldice', 'id': '5152439', 'version': '::apt', 'url': 'https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463', 'action': 'install'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "rolldice",
            "action": "install",
            "id": "5152439",
            "version": "::apt",
            "url": "https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463"
        }
    ]
}
10:59:36 INFO  Started operation: {'creationTime': '2022-05-02T10:59:36.182Z', 'deviceName': 'm32sd10c', 'deviceId': '18127529', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18152682', 'id': '18152682', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'name': 'rolldice', 'id': '5152439', 'version': '::apt', 'url': 'https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463', 'action': 'install'}]", 'c8y_SoftwareUpdate': [{'name': 'rolldice', 'action': 'install', 'id': '5152439', 'version': '::apt', 'url': 'https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463'}]}
10:59:36 WARN  , retrying in 0.5 seconds...
10:59:36 WARN  , retrying in 0.5 seconds...
10:59:37 WARN  , retrying in 0.5 seconds...
10:59:37 WARN  , retrying in 0.5 seconds...
10:59:38 WARN  , retrying in 0.5 seconds...
10:59:38 WARN  , retrying in 0.5 seconds...
10:59:39 WARN  , retrying in 0.5 seconds...
10:59:39 WARN  , retrying in 0.5 seconds...
10:59:40 WARN  , retrying in 0.5 seconds...
10:59:40 WARN  AssertionError occurred while running test - 
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1294, in __call__
    self.testObj.execute()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/run.py", line 83, in execute
    assert_install_in_download_path(install_directory=self.DOWNLOAD_DIR)
  File "/home/pi/env-pysys/lib/python3.7/site-packages/decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
  File "/home/pi/env-pysys/lib/python3.7/site-packages/retry/api.py", line 74, in retry_decorator
    logger)
  File "/home/pi/env-pysys/lib/python3.7/site-packages/retry/api.py", line 33, in __retry_internal
    return f()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/run.py", line 28, in assert_install_in_download_path
    assert "rolldice" in os.listdir(f"{install_directory}")
AssertionError
10:59:41 WARN  AssertionError: ... blocked 
10:59:41 INFO  
10:59:41 INFO  cleanup:
10:59:41 INFO  Executed sudo, exit status 0
10:59:42 INFO  Executed sudo, exit status 0
10:59:47 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
10:59:48 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 5 seconds
10:59:48 INFO  Delaying execution by 6 seconds
11:00:08 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 13 secs
11:00:12 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
11:00:13 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
11:00:13 INFO  
11:00:13 INFO  Test duration: 85.61 secs
11:00:13 INFO  Test final outcome:  BLOCKED
11:00:13 INFO  Test outcome reason: AssertionError:
11:00:13 INFO  
</system-out>
	</testcase>
<testcase classname="AptInstallWithDownloadPath" name="SoftwareManagement.sm_apt_install_download_path">
		<failure message="BLOCKED: AssertionError:" type="BLOCKED" />
		<system-out>11:24:15 INFO  ==============================================================
11:24:15 INFO  Id:    SoftwareManagement.sm_apt_install_download_path
11:24:15 INFO  Dir:   software_management_end_to_end/
11:24:15 INFO  Title: Install package to directory specified in tedge config set tmp.path
11:24:15 INFO  ==============================================================
11:24:16 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
11:24:16 INFO  EnvironmentC8y Setup
11:24:17 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
11:24:31 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 13 secs
11:24:31 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
11:24:32 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
11:24:32 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
11:24:34 INFO  State of current operation: FAILED
11:24:35 INFO  Assert that (False == value) with value=False ... passed
11:24:35 INFO  Executed sudo, exit status 0
11:24:36 INFO  Executed sudo, exit status 0
11:24:36 INFO  Executed sudo, exit status 0
11:24:40 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
11:24:40 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 3 seconds
11:24:40 INFO  Delaying execution by 8 seconds
11:25:03 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
11:25:03 INFO  Response status: 201
11:25:03 INFO  Response to action: {
    "creationTime": "2022-05-02T11:25:03.930Z",
    "deviceName": "m32sd10c",
    "deviceId": "18127529",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18153040",
    "id": "18153040",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'name': 'rolldice', 'id': '5152439', 'version': '::apt', 'url': 'https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463', 'action': 'install'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "rolldice",
            "action": "install",
            "id": "5152439",
            "version": "::apt",
            "url": "https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463"
        }
    ]
}
11:25:03 INFO  Started operation: {'creationTime': '2022-05-02T11:25:03.930Z', 'deviceName': 'm32sd10c', 'deviceId': '18127529', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18153040', 'id': '18153040', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'name': 'rolldice', 'id': '5152439', 'version': '::apt', 'url': 'https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463', 'action': 'install'}]", 'c8y_SoftwareUpdate': [{'name': 'rolldice', 'action': 'install', 'id': '5152439', 'version': '::apt', 'url': 'https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463'}]}
11:25:03 WARN  , retrying in 0.5 seconds...
11:25:04 WARN  , retrying in 0.5 seconds...
11:25:04 WARN  , retrying in 0.5 seconds...
11:25:05 WARN  , retrying in 0.5 seconds...
11:25:05 WARN  , retrying in 0.5 seconds...
11:25:06 WARN  , retrying in 0.5 seconds...
11:25:07 WARN  , retrying in 0.5 seconds...
11:25:07 WARN  , retrying in 0.5 seconds...
11:25:08 WARN  , retrying in 0.5 seconds...
11:25:08 WARN  AssertionError occurred while running test - 
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1294, in __call__
    self.testObj.execute()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/run.py", line 83, in execute
    assert_install_in_download_path(install_directory=self.DOWNLOAD_DIR)
  File "/home/pi/env-pysys/lib/python3.7/site-packages/decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
  File "/home/pi/env-pysys/lib/python3.7/site-packages/retry/api.py", line 74, in retry_decorator
    logger)
  File "/home/pi/env-pysys/lib/python3.7/site-packages/retry/api.py", line 33, in __retry_internal
    return f()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/run.py", line 28, in assert_install_in_download_path
    assert "rolldice" in os.listdir(f"{install_directory}")
AssertionError
11:25:08 WARN  AssertionError: ... blocked 
11:25:08 INFO  
11:25:08 INFO  cleanup:
11:25:09 INFO  Executed sudo, exit status 0
11:25:09 INFO  Executed sudo, exit status 0
11:25:13 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
11:25:13 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 4 seconds
11:25:13 INFO  Delaying execution by 7 seconds
11:25:34 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 13 secs
11:25:39 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
11:25:39 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
11:25:39 INFO  
11:25:39 INFO  Test duration: 84.52 secs
11:25:39 INFO  Test final outcome:  BLOCKED
11:25:39 INFO  Test outcome reason: AssertionError:
11:25:39 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="SoftwareManagement.sm_apt_install_fail" tests="3" errors="0" failures="0" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_fail">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it" />
		<system-out>10:34:23 INFO  ==============================================================
10:34:23 INFO  Id:    SoftwareManagement.sm_apt_install_fail
10:34:23 INFO  Dir:   software_management_end_to_end/
10:34:23 INFO  Title: Validate end to end behaviour for a failing installation
10:34:23 INFO  ==============================================================
10:34:23 INFO  Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it ... skipped
10:34:23 WARN  Aborted test due to SKIPPED outcome
10:34:23 INFO  
10:34:23 INFO  Test duration: 0.02 secs
10:34:23 INFO  Test final outcome:  SKIPPED
10:34:23 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it
10:34:23 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_fail" />
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_fail" />
</testsuite><testsuite name="SoftwareManagement.sm_apt_install_remove" tests="3" errors="0" failures="0" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it" />
		<system-out>10:34:23 INFO  ==============================================================
10:34:23 INFO  Id:    SoftwareManagement.sm_apt_install_remove
10:34:23 INFO  Dir:   software_management_end_to_end/
10:34:23 INFO  Title: Validate end to end behaviour for the apt plugin for installation and removal of packages
10:34:23 INFO  ==============================================================
10:34:23 INFO  Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it ... skipped
10:34:23 WARN  Aborted test due to SKIPPED outcome
10:34:23 INFO  
10:34:23 INFO  Test duration: 0.02 secs
10:34:23 INFO  Test final outcome:  SKIPPED
10:34:23 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it
10:34:23 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove" />
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove" />
</testsuite><testsuite name="SoftwareManagement.sm_apt_install_remove_default" tests="3" errors="0" failures="0" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_default">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it" />
		<system-out>10:34:23 INFO  ==============================================================
10:34:23 INFO  Id:    SoftwareManagement.sm_apt_install_remove_default
10:34:23 INFO  Dir:   software_management_end_to_end/
10:34:23 INFO  Title: Validate end to end behaviour for the default plugin for installation and removal of packages
10:34:23 INFO  ==============================================================
10:34:23 INFO  Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it ... skipped
10:34:23 WARN  Aborted test due to SKIPPED outcome
10:34:23 INFO  
10:34:23 INFO  Test duration: 0.02 secs
10:34:23 INFO  Test final outcome:  SKIPPED
10:34:23 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it
10:34:23 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_default" />
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_default" />
</testsuite><testsuite name="SoftwareManagement.sm_apt_install_remove_multiple" tests="3" errors="0" failures="2" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_multiple">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it" />
		<system-out>10:34:23 INFO  ==============================================================
10:34:23 INFO  Id:    SoftwareManagement.sm_apt_install_remove_multiple
10:34:23 INFO  Dir:   software_management_end_to_end/
10:34:23 INFO  Title: Validate end to end behaviour for the apt plugin for multiple packages
10:34:23 INFO  ==============================================================
10:34:23 INFO  Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it ... skipped
10:34:23 WARN  Aborted test due to SKIPPED outcome
10:34:23 INFO  
10:34:23 INFO  Test duration: 0.02 secs
10:34:23 INFO  Test final outcome:  SKIPPED
10:34:23 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it
10:34:23 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_multiple">
		<failure message="FAILED: Assert that (False == value) with value=True" type="FAILED" />
		<system-out>11:07:10 INFO  ==============================================================
11:07:10 INFO  Id:    SoftwareManagement.sm_apt_install_remove_multiple
11:07:10 INFO  Dir:   software_management_end_to_end/
11:07:10 INFO  Title: Validate end to end behaviour for the apt plugin for multiple packages
11:07:10 INFO  ==============================================================
11:07:10 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
11:07:10 INFO  EnvironmentC8y Setup
11:07:10 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
11:07:11 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 5 seconds
11:07:11 INFO  Delaying execution by 6 seconds
11:07:31 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
11:07:32 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
11:07:32 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
11:07:33 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
11:07:35 INFO  State of current operation: SUCCESSFUL
11:07:35 INFO  Assert that (False == value) with value=False ... passed
11:07:36 INFO  Assert that (False == value) with value=False ... passed
11:07:36 INFO  Assert that (False == value) with value=False ... passed
11:07:36 INFO  Package robotfindskitten is installed
11:07:36 WARN  Assert that (False == value) with value=True ... failed 
11:07:36 INFO  Response status: 201
11:07:36 INFO  Response to action: {
    "creationTime": "2022-05-02T11:07:36.687Z",
    "deviceName": "m32sd10c",
    "deviceId": "18127529",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18152800",
    "id": "18152800",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '::apt'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "asciijump",
            "action": "install",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "robotfindskitten",
            "action": "install",
            "id": "5474869",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "squirrel3",
            "action": "install",
            "id": "5475279",
            "version": "::apt",
            "url": " "
        }
    ]
}
11:07:36 INFO  Started operation: {'creationTime': '2022-05-02T11:07:36.687Z', 'deviceName': 'm32sd10c', 'deviceId': '18127529', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18152800', 'id': '18152800', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '::apt'}]", 'c8y_SoftwareUpdate': [{'name': 'asciijump', 'action': 'install', 'id': '5475369', 'version': '::apt', 'url': ' '}, {'name': 'robotfindskitten', 'action': 'install', 'id': '5474869', 'version': '::apt', 'url': ' '}, {'name': 'squirrel3', 'action': 'install', 'id': '5475279', 'version': '::apt', 'url': ' '}]}
11:07:38 INFO  State of operation 18152800 : PENDING
11:07:41 INFO  State of operation 18152800 : EXECUTING
11:07:43 INFO  State of operation 18152800 : EXECUTING
11:07:45 INFO  State of operation 18152800 : EXECUTING
11:07:47 INFO  State of operation 18152800 : EXECUTING
11:07:50 INFO  State of operation 18152800 : EXECUTING
11:07:52 INFO  State of operation 18152800 : EXECUTING
11:07:55 INFO  State of operation 18152800 : EXECUTING
11:07:58 INFO  State of operation 18152800 : EXECUTING
11:08:00 INFO  State of operation 18152800 : EXECUTING
11:08:03 INFO  State of operation 18152800 : EXECUTING
11:08:05 INFO  State of operation 18152800 : EXECUTING
11:08:07 INFO  State of operation 18152800 : EXECUTING
11:08:09 INFO  State of operation 18152800 : EXECUTING
11:08:12 INFO  State of operation 18152800 : EXECUTING
11:08:14 INFO  State of operation 18152800 : EXECUTING
11:08:16 INFO  State of operation 18152800 : EXECUTING
11:08:19 INFO  State of operation 18152800 : EXECUTING
11:08:21 INFO  State of operation 18152800 : EXECUTING
11:08:24 INFO  State of operation 18152800 : EXECUTING
11:08:26 INFO  State of operation 18152800 : EXECUTING
11:08:28 INFO  State of operation 18152800 : EXECUTING
11:08:31 INFO  State of operation 18152800 : EXECUTING
11:08:33 INFO  State of operation 18152800 : EXECUTING
11:08:36 INFO  State of operation 18152800 : EXECUTING
11:08:38 INFO  State of operation 18152800 : EXECUTING
11:08:41 INFO  State of operation 18152800 : EXECUTING
11:08:44 INFO  State of operation 18152800 : EXECUTING
11:08:46 INFO  State of operation 18152800 : EXECUTING
11:08:49 INFO  State of operation 18152800 : EXECUTING
11:08:51 INFO  State of operation 18152800 : EXECUTING
11:08:53 INFO  State of operation 18152800 : EXECUTING
11:08:55 INFO  State of operation 18152800 : EXECUTING
11:08:58 INFO  State of operation 18152800 : EXECUTING
11:09:00 INFO  State of operation 18152800 : SUCCESSFUL
11:09:00 INFO  Assert that (False == value) with value=False ... passed
11:09:00 INFO  Package asciijump is installed
11:09:00 INFO  Assert that (True == value) with value=True ... passed
11:09:01 INFO  Package squirrel3 is installed
11:09:01 INFO  Assert that (True == value) with value=True ... passed
11:09:01 INFO  Package robotfindskitten is installed
11:09:01 INFO  Assert that (True == value) with value=True ... passed
11:09:01 INFO  Response status: 201
11:09:01 INFO  Response to action: {
    "creationTime": "2022-05-02T11:09:01.458Z",
    "deviceName": "m32sd10c",
    "deviceId": "18127529",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18152823",
    "id": "18152823",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'delete', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'delete', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': '::apt'}, {'action': 'delete', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '::apt'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "asciijump",
            "action": "delete",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "robotfindskitten",
            "action": "delete",
            "id": "5474869",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "squirrel3",
            "action": "delete",
            "id": "5475279",
            "version": "::apt",
            "url": " "
        }
    ]
}
11:09:01 INFO  Started operation: {'creationTime': '2022-05-02T11:09:01.458Z', 'deviceName': 'm32sd10c', 'deviceId': '18127529', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18152823', 'id': '18152823', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'delete', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'delete', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': '::apt'}, {'action': 'delete', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '::apt'}]", 'c8y_SoftwareUpdate': [{'name': 'asciijump', 'action': 'delete', 'id': '5475369', 'version': '::apt', 'url': ' '}, {'name': 'robotfindskitten', 'action': 'delete', 'id': '5474869', 'version': '::apt', 'url': ' '}, {'name': 'squirrel3', 'action': 'delete', 'id': '5475279', 'version': '::apt', 'url': ' '}]}
11:09:03 INFO  State of operation 18152823 : PENDING
11:09:06 INFO  State of operation 18152823 : EXECUTING
11:09:09 INFO  State of operation 18152823 : EXECUTING
11:09:11 INFO  State of operation 18152823 : EXECUTING
11:09:13 INFO  State of operation 18152823 : EXECUTING
11:09:16 INFO  State of operation 18152823 : EXECUTING
11:09:19 INFO  State of operation 18152823 : EXECUTING
11:09:21 INFO  State of operation 18152823 : EXECUTING
11:09:24 INFO  State of operation 18152823 : EXECUTING
11:09:26 INFO  State of operation 18152823 : EXECUTING
11:09:28 INFO  State of operation 18152823 : EXECUTING
11:09:31 INFO  State of operation 18152823 : EXECUTING
11:09:33 INFO  State of operation 18152823 : EXECUTING
11:09:35 INFO  State of operation 18152823 : EXECUTING
11:09:37 INFO  State of operation 18152823 : EXECUTING
11:09:40 INFO  State of operation 18152823 : EXECUTING
11:09:42 INFO  State of operation 18152823 : EXECUTING
11:09:45 INFO  State of operation 18152823 : EXECUTING
11:09:47 INFO  State of operation 18152823 : EXECUTING
11:09:50 INFO  State of operation 18152823 : EXECUTING
11:09:52 INFO  State of operation 18152823 : EXECUTING
11:09:54 INFO  State of operation 18152823 : EXECUTING
11:09:56 INFO  State of operation 18152823 : EXECUTING
11:09:59 INFO  State of operation 18152823 : EXECUTING
11:10:01 INFO  State of operation 18152823 : EXECUTING
11:10:03 INFO  State of operation 18152823 : EXECUTING
11:10:06 INFO  State of operation 18152823 : EXECUTING
11:10:08 INFO  State of operation 18152823 : EXECUTING
11:10:11 INFO  State of operation 18152823 : EXECUTING
11:10:13 INFO  State of operation 18152823 : EXECUTING
11:10:15 INFO  State of operation 18152823 : SUCCESSFUL
11:10:16 INFO  Assert that (False == value) with value=False ... passed
11:10:16 INFO  Assert that (False == value) with value=False ... passed
11:10:16 INFO  Assert that (False == value) with value=False ... passed
11:10:16 INFO  Assert that (False == value) with value=False ... passed
11:10:16 INFO  
11:10:16 INFO  cleanup:
11:10:20 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
11:10:21 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
11:10:21 INFO  
11:10:21 INFO  Test duration: 191.07 secs
11:10:21 INFO  Test final outcome:  FAILED
11:10:21 INFO  Test outcome reason: Assert that (False == value) with value=True
11:10:21 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_multiple">
		<failure message="FAILED: Assert that (False == value) with value=True (+2 other failures)" type="FAILED" />
		<system-out>11:32:26 INFO  ==============================================================
11:32:26 INFO  Id:    SoftwareManagement.sm_apt_install_remove_multiple
11:32:26 INFO  Dir:   software_management_end_to_end/
11:32:26 INFO  Title: Validate end to end behaviour for the apt plugin for multiple packages
11:32:26 INFO  ==============================================================
11:32:26 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
11:32:26 INFO  EnvironmentC8y Setup
11:32:26 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
11:32:27 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 5 seconds
11:32:27 INFO  Delaying execution by 6 seconds
11:32:47 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
11:32:47 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
11:32:48 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
11:32:48 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
11:32:50 INFO  State of current operation: SUCCESSFUL
11:32:51 INFO  Assert that (False == value) with value=False ... passed
11:32:51 INFO  Package asciijump is installed
11:32:51 WARN  Assert that (False == value) with value=True ... failed 
11:32:51 INFO  Package squirrel3 is installed
11:32:52 WARN  Assert that (False == value) with value=True ... failed 
11:32:52 INFO  Package robotfindskitten is installed
11:32:52 WARN  Assert that (False == value) with value=True ... failed 
11:32:52 INFO  Response status: 201
11:32:52 INFO  Response to action: {
    "creationTime": "2022-05-02T11:32:52.450Z",
    "deviceName": "m32sd10c",
    "deviceId": "18127529",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18168127",
    "id": "18168127",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '::apt'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "asciijump",
            "action": "install",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "robotfindskitten",
            "action": "install",
            "id": "5474869",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "squirrel3",
            "action": "install",
            "id": "5475279",
            "version": "::apt",
            "url": " "
        }
    ]
}
11:32:52 INFO  Started operation: {'creationTime': '2022-05-02T11:32:52.450Z', 'deviceName': 'm32sd10c', 'deviceId': '18127529', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18168127', 'id': '18168127', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '::apt'}]", 'c8y_SoftwareUpdate': [{'name': 'asciijump', 'action': 'install', 'id': '5475369', 'version': '::apt', 'url': ' '}, {'name': 'robotfindskitten', 'action': 'install', 'id': '5474869', 'version': '::apt', 'url': ' '}, {'name': 'squirrel3', 'action': 'install', 'id': '5475279', 'version': '::apt', 'url': ' '}]}
11:32:54 INFO  State of operation 18168127 : PENDING
11:32:56 INFO  State of operation 18168127 : EXECUTING
11:32:59 INFO  State of operation 18168127 : EXECUTING
11:33:01 INFO  State of operation 18168127 : EXECUTING
11:33:03 INFO  State of operation 18168127 : EXECUTING
11:33:05 INFO  State of operation 18168127 : EXECUTING
11:33:09 INFO  State of operation 18168127 : EXECUTING
11:33:12 INFO  State of operation 18168127 : EXECUTING
11:33:14 INFO  State of operation 18168127 : EXECUTING
11:33:17 INFO  State of operation 18168127 : EXECUTING
11:33:19 INFO  State of operation 18168127 : EXECUTING
11:33:21 INFO  State of operation 18168127 : EXECUTING
11:33:23 INFO  State of operation 18168127 : EXECUTING
11:33:26 INFO  State of operation 18168127 : EXECUTING
11:33:28 INFO  State of operation 18168127 : EXECUTING
11:33:30 INFO  State of operation 18168127 : SUCCESSFUL
11:33:30 INFO  Assert that (False == value) with value=False ... passed
11:33:31 INFO  Package asciijump is installed
11:33:31 INFO  Assert that (True == value) with value=True ... passed
11:33:31 INFO  Package squirrel3 is installed
11:33:31 INFO  Assert that (True == value) with value=True ... passed
11:33:31 INFO  Package robotfindskitten is installed
11:33:31 INFO  Assert that (True == value) with value=True ... passed
11:33:31 INFO  Response status: 201
11:33:31 INFO  Response to action: {
    "creationTime": "2022-05-02T11:33:31.672Z",
    "deviceName": "m32sd10c",
    "deviceId": "18127529",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18168138",
    "id": "18168138",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'delete', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'delete', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': '::apt'}, {'action': 'delete', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '::apt'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "asciijump",
            "action": "delete",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "robotfindskitten",
            "action": "delete",
            "id": "5474869",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "squirrel3",
            "action": "delete",
            "id": "5475279",
            "version": "::apt",
            "url": " "
        }
    ]
}
11:33:31 INFO  Started operation: {'creationTime': '2022-05-02T11:33:31.672Z', 'deviceName': 'm32sd10c', 'deviceId': '18127529', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18168138', 'id': '18168138', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'delete', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'delete', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': '::apt'}, {'action': 'delete', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '::apt'}]", 'c8y_SoftwareUpdate': [{'name': 'asciijump', 'action': 'delete', 'id': '5475369', 'version': '::apt', 'url': ' '}, {'name': 'robotfindskitten', 'action': 'delete', 'id': '5474869', 'version': '::apt', 'url': ' '}, {'name': 'squirrel3', 'action': 'delete', 'id': '5475279', 'version': '::apt', 'url': ' '}]}
11:33:33 INFO  State of operation 18168138 : PENDING
11:33:36 INFO  State of operation 18168138 : EXECUTING
11:33:38 INFO  State of operation 18168138 : EXECUTING
11:33:40 INFO  State of operation 18168138 : EXECUTING
11:33:42 INFO  State of operation 18168138 : EXECUTING
11:33:45 INFO  State of operation 18168138 : EXECUTING
11:33:47 INFO  State of operation 18168138 : EXECUTING
11:33:50 INFO  State of operation 18168138 : EXECUTING
11:33:52 INFO  State of operation 18168138 : EXECUTING
11:33:55 INFO  State of operation 18168138 : EXECUTING
11:33:57 INFO  State of operation 18168138 : EXECUTING
11:33:59 INFO  State of operation 18168138 : EXECUTING
11:34:01 INFO  State of operation 18168138 : EXECUTING
11:34:04 INFO  State of operation 18168138 : EXECUTING
11:34:06 INFO  State of operation 18168138 : EXECUTING
11:34:08 INFO  State of operation 18168138 : EXECUTING
11:34:10 INFO  State of operation 18168138 : EXECUTING
11:34:13 INFO  State of operation 18168138 : EXECUTING
11:34:15 INFO  State of operation 18168138 : EXECUTING
11:34:18 INFO  State of operation 18168138 : EXECUTING
11:34:20 INFO  State of operation 18168138 : EXECUTING
11:34:22 INFO  State of operation 18168138 : EXECUTING
11:34:25 INFO  State of operation 18168138 : EXECUTING
11:34:27 INFO  State of operation 18168138 : EXECUTING
11:34:29 INFO  State of operation 18168138 : EXECUTING
11:34:32 INFO  State of operation 18168138 : EXECUTING
11:34:34 INFO  State of operation 18168138 : EXECUTING
11:34:36 INFO  State of operation 18168138 : EXECUTING
11:34:39 INFO  State of operation 18168138 : EXECUTING
11:34:41 INFO  State of operation 18168138 : SUCCESSFUL
11:34:41 INFO  Assert that (False == value) with value=False ... passed
11:34:41 INFO  Assert that (False == value) with value=False ... passed
11:34:42 INFO  Assert that (False == value) with value=False ... passed
11:34:42 INFO  Assert that (False == value) with value=False ... passed
11:34:42 INFO  
11:34:42 INFO  cleanup:
11:34:46 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
11:34:46 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
11:34:46 INFO  
11:34:46 INFO  Test duration: 140.54 secs
11:34:46 INFO  Test final outcome:  FAILED
11:34:46 INFO  Test outcome reason: Assert that (False == value) with value=True (+2 other failures)
11:34:46 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="SoftwareManagement.sm_apt_install_remove_multiple_mixed" tests="3" errors="0" failures="2" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_multiple_mixed">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it" />
		<system-out>10:34:23 INFO  ==============================================================
10:34:23 INFO  Id:    SoftwareManagement.sm_apt_install_remove_multiple_mixed
10:34:23 INFO  Dir:   software_management_end_to_end/
10:34:23 INFO  Title: Validate end to end behaviour for the apt plugin for multiple packages with mixed versions
10:34:23 INFO  ==============================================================
10:34:23 INFO  Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it ... skipped
10:34:23 WARN  Aborted test due to SKIPPED outcome
10:34:23 INFO  
10:34:23 INFO  Test duration: 0.02 secs
10:34:23 INFO  Test final outcome:  SKIPPED
10:34:23 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it
10:34:23 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_multiple_mixed">
		<failure message="BLOCKED: SystemError: Timeout while waiting for status SUCCESSFUL or False" type="BLOCKED" />
		<system-out>11:10:21 INFO  ==============================================================
11:10:21 INFO  Id:    SoftwareManagement.sm_apt_install_remove_multiple_mixed
11:10:21 INFO  Dir:   software_management_end_to_end/
11:10:21 INFO  Title: Validate end to end behaviour for the apt plugin for multiple packages with mixed versions
11:10:21 INFO  ==============================================================
11:10:21 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
11:10:21 INFO  EnvironmentC8y Setup
11:10:22 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
11:10:22 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 5 seconds
11:10:22 INFO  Delaying execution by 6 seconds
11:10:42 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 13 secs
11:10:42 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
11:10:43 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
11:10:43 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
11:10:45 INFO  State of current operation: SUCCESSFUL
11:10:46 INFO  Assert that (False == value) with value=False ... passed
11:10:46 INFO  Assert that (False == value) with value=False ... passed
11:10:46 INFO  Assert that (False == value) with value=False ... passed
11:10:46 INFO  Assert that (False == value) with value=False ... passed
11:10:46 INFO  Assert that (False == value) with value=False ... passed
11:10:48 INFO  Response status: 201
11:10:48 INFO  Response to action: {
    "creationTime": "2022-05-02T11:10:48.189Z",
    "deviceName": "m32sd10c",
    "deviceId": "18127529",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18152849",
    "id": "18152849",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': ' '}, {'action': 'install', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '3.1-7::apt'}, {'action': 'install', 'id': '5152439', 'name': 'rolldice', 'url': ' ', 'version': '1.16-1+b1'}, {'action': 'install', 'id': '5439204', 'name': 'moon-buggy', 'url': ' ', 'version': '1:1.0.51-12'}, {'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "asciijump",
            "action": "install",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "robotfindskitten",
            "action": "install",
            "id": "5474869",
            "version": " ",
            "url": " "
        },
        {
            "name": "squirrel3",
            "action": "install",
            "id": "5475279",
            "version": "3.1-7::apt",
            "url": " "
        },
        {
            "name": "rolldice",
            "action": "install",
            "id": "5152439",
            "version": "1.16-1+b1",
            "url": " "
        },
        {
            "name": "moon-buggy",
            "action": "install",
            "id": "5439204",
            "version": "1:1.0.51-12",
            "url": " "
        },
        {
            "name": "asciijump",
            "action": "install",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        }
    ]
}
11:10:48 INFO  Started operation: {'creationTime': '2022-05-02T11:10:48.189Z', 'deviceName': 'm32sd10c', 'deviceId': '18127529', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18152849', 'id': '18152849', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': ' '}, {'action': 'install', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '3.1-7::apt'}, {'action': 'install', 'id': '5152439', 'name': 'rolldice', 'url': ' ', 'version': '1.16-1+b1'}, {'action': 'install', 'id': '5439204', 'name': 'moon-buggy', 'url': ' ', 'version': '1:1.0.51-12'}, {'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}]", 'c8y_SoftwareUpdate': [{'name': 'asciijump', 'action': 'install', 'id': '5475369', 'version': '::apt', 'url': ' '}, {'name': 'robotfindskitten', 'action': 'install', 'id': '5474869', 'version': ' ', 'url': ' '}, {'name': 'squirrel3', 'action': 'install', 'id': '5475279', 'version': '3.1-7::apt', 'url': ' '}, {'name': 'rolldice', 'action': 'install', 'id': '5152439', 'version': '1.16-1+b1', 'url': ' '}, {'name': 'moon-buggy', 'action': 'install', 'id': '5439204', 'version': '1:1.0.51-12', 'url': ' '}, {'name': 'asciijump', 'action': 'install', 'id': '5475369', 'version': '::apt', 'url': ' '}]}
11:10:50 INFO  State of operation 18152849 : PENDING
11:10:52 INFO  State of operation 18152849 : EXECUTING
11:10:54 INFO  State of operation 18152849 : EXECUTING
11:10:57 INFO  State of operation 18152849 : EXECUTING
11:10:59 INFO  State of operation 18152849 : EXECUTING
11:11:01 INFO  State of operation 18152849 : EXECUTING
11:11:04 INFO  State of operation 18152849 : EXECUTING
11:11:07 INFO  State of operation 18152849 : EXECUTING
11:11:10 INFO  State of operation 18152849 : EXECUTING
11:11:12 INFO  State of operation 18152849 : EXECUTING
11:11:14 INFO  State of operation 18152849 : EXECUTING
11:11:17 INFO  State of operation 18152849 : EXECUTING
11:11:19 INFO  State of operation 18152849 : EXECUTING
11:11:21 INFO  State of operation 18152849 : EXECUTING
11:11:23 INFO  State of operation 18152849 : EXECUTING
11:11:26 INFO  State of operation 18152849 : EXECUTING
11:11:28 INFO  State of operation 18152849 : EXECUTING
11:11:31 INFO  State of operation 18152849 : EXECUTING
11:11:33 INFO  State of operation 18152849 : EXECUTING
11:11:35 INFO  State of operation 18152849 : EXECUTING
11:11:37 INFO  State of operation 18152849 : EXECUTING
11:11:40 INFO  State of operation 18152849 : EXECUTING
11:11:42 INFO  State of operation 18152849 : EXECUTING
11:11:45 INFO  State of operation 18152849 : EXECUTING
11:11:47 INFO  State of operation 18152849 : EXECUTING
11:11:50 INFO  State of operation 18152849 : EXECUTING
11:11:52 INFO  State of operation 18152849 : EXECUTING
11:11:54 INFO  State of operation 18152849 : EXECUTING
11:11:56 INFO  State of operation 18152849 : EXECUTING
11:11:59 INFO  State of operation 18152849 : EXECUTING
11:12:01 INFO  State of operation 18152849 : EXECUTING
11:12:03 INFO  State of operation 18152849 : EXECUTING
11:12:06 INFO  State of operation 18152849 : EXECUTING
11:12:08 INFO  State of operation 18152849 : EXECUTING
11:12:10 INFO  State of operation 18152849 : EXECUTING
11:12:12 INFO  State of operation 18152849 : EXECUTING
11:12:15 INFO  State of operation 18152849 : EXECUTING
11:12:17 INFO  State of operation 18152849 : EXECUTING
11:12:20 INFO  State of operation 18152849 : EXECUTING
11:12:22 INFO  State of operation 18152849 : EXECUTING
11:12:25 INFO  State of operation 18152849 : EXECUTING
11:12:27 INFO  State of operation 18152849 : EXECUTING
11:12:29 INFO  State of operation 18152849 : EXECUTING
11:12:31 INFO  State of operation 18152849 : EXECUTING
11:12:34 INFO  State of operation 18152849 : EXECUTING
11:12:36 INFO  State of operation 18152849 : EXECUTING
11:12:38 INFO  State of operation 18152849 : EXECUTING
11:12:40 INFO  State of operation 18152849 : EXECUTING
11:12:43 INFO  State of operation 18152849 : EXECUTING
11:12:45 INFO  State of operation 18152849 : EXECUTING
11:12:48 INFO  State of operation 18152849 : EXECUTING
11:12:50 INFO  State of operation 18152849 : EXECUTING
11:12:52 INFO  State of operation 18152849 : EXECUTING
11:12:54 INFO  State of operation 18152849 : EXECUTING
11:12:57 INFO  State of operation 18152849 : EXECUTING
11:12:59 INFO  State of operation 18152849 : EXECUTING
11:13:01 INFO  State of operation 18152849 : EXECUTING
11:13:04 INFO  State of operation 18152849 : EXECUTING
11:13:07 INFO  State of operation 18152849 : EXECUTING
11:13:09 INFO  State of operation 18152849 : EXECUTING
11:13:12 INFO  State of operation 18152849 : EXECUTING
11:13:14 WARN  SystemError occurred while running test - Timeout while waiting for status SUCCESSFUL or False
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1294, in __call__
    self.testObj.execute()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_remove_multiple_mixed/run.py", line 89, in execute
    self.wait_until_succcess()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 324, in wait_until_succcess
    self.wait_until_status("SUCCESSFUL")
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 381, in wait_until_status
    "Timeout while waiting for status %s or %s" % (status, status2)
SystemError: Timeout while waiting for status SUCCESSFUL or False
11:13:14 WARN  SystemError: Timeout while waiting for status SUCCESSFUL or False ... blocked 
11:13:14 INFO  
11:13:14 INFO  cleanup:
11:13:20 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
11:13:21 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
11:13:21 INFO  
11:13:21 INFO  Test duration: 180.07 secs
11:13:21 INFO  Test final outcome:  BLOCKED
11:13:21 INFO  Test outcome reason: SystemError: Timeout while waiting for status SUCCESSFUL or False
11:13:21 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_multiple_mixed">
		<failure message="BLOCKED: SystemError: Timeout while waiting for status SUCCESSFUL or False (+1 other failures)" type="BLOCKED" />
		<system-out>11:34:46 INFO  ==============================================================
11:34:46 INFO  Id:    SoftwareManagement.sm_apt_install_remove_multiple_mixed
11:34:46 INFO  Dir:   software_management_end_to_end/
11:34:46 INFO  Title: Validate end to end behaviour for the apt plugin for multiple packages with mixed versions
11:34:46 INFO  ==============================================================
11:34:47 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
11:34:47 INFO  EnvironmentC8y Setup
11:34:47 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
11:34:47 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 4 seconds
11:34:47 INFO  Delaying execution by 7 seconds
11:35:08 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
11:35:09 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
11:35:10 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
11:35:10 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
11:35:12 INFO  State of current operation: SUCCESSFUL
11:35:12 INFO  Assert that (False == value) with value=False ... passed
11:35:13 INFO  Assert that (False == value) with value=False ... passed
11:35:13 INFO  Assert that (False == value) with value=False ... passed
11:35:13 INFO  Assert that (False == value) with value=False ... passed
11:35:13 INFO  Package moon-buggy is installed
11:35:13 WARN  Assert that (False == value) with value=True ... failed 
11:35:15 INFO  Response status: 201
11:35:15 INFO  Response to action: {
    "delivery": {
        "time": "2022-05-02T11:35:15.229Z",
        "log": [],
        "status": "PENDING"
    },
    "creationTime": "2022-05-02T11:35:15.216Z",
    "deviceName": "m32sd10c",
    "deviceId": "18127529",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18168151",
    "id": "18168151",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': ' '}, {'action': 'install', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '3.1-7::apt'}, {'action': 'install', 'id': '5152439', 'name': 'rolldice', 'url': ' ', 'version': '1.16-1+b1'}, {'action': 'install', 'id': '5439204', 'name': 'moon-buggy', 'url': ' ', 'version': '1:1.0.51-12'}, {'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "asciijump",
            "action": "install",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "robotfindskitten",
            "action": "install",
            "id": "5474869",
            "version": " ",
            "url": " "
        },
        {
            "name": "squirrel3",
            "action": "install",
            "id": "5475279",
            "version": "3.1-7::apt",
            "url": " "
        },
        {
            "name": "rolldice",
            "action": "install",
            "id": "5152439",
            "version": "1.16-1+b1",
            "url": " "
        },
        {
            "name": "moon-buggy",
            "action": "install",
            "id": "5439204",
            "version": "1:1.0.51-12",
            "url": " "
        },
        {
            "name": "asciijump",
            "action": "install",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        }
    ]
}
11:35:15 INFO  Started operation: {'delivery': {'time': '2022-05-02T11:35:15.229Z', 'log': [], 'status': 'PENDING'}, 'creationTime': '2022-05-02T11:35:15.216Z', 'deviceName': 'm32sd10c', 'deviceId': '18127529', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18168151', 'id': '18168151', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': ' '}, {'action': 'install', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '3.1-7::apt'}, {'action': 'install', 'id': '5152439', 'name': 'rolldice', 'url': ' ', 'version': '1.16-1+b1'}, {'action': 'install', 'id': '5439204', 'name': 'moon-buggy', 'url': ' ', 'version': '1:1.0.51-12'}, {'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}]", 'c8y_SoftwareUpdate': [{'name': 'asciijump', 'action': 'install', 'id': '5475369', 'version': '::apt', 'url': ' '}, {'name': 'robotfindskitten', 'action': 'install', 'id': '5474869', 'version': ' ', 'url': ' '}, {'name': 'squirrel3', 'action': 'install', 'id': '5475279', 'version': '3.1-7::apt', 'url': ' '}, {'name': 'rolldice', 'action': 'install', 'id': '5152439', 'version': '1.16-1+b1', 'url': ' '}, {'name': 'moon-buggy', 'action': 'install', 'id': '5439204', 'version': '1:1.0.51-12', 'url': ' '}, {'name': 'asciijump', 'action': 'install', 'id': '5475369', 'version': '::apt', 'url': ' '}]}
11:35:17 INFO  State of operation 18168151 : PENDING
11:35:19 INFO  State of operation 18168151 : EXECUTING
11:35:21 INFO  State of operation 18168151 : EXECUTING
11:35:24 INFO  State of operation 18168151 : EXECUTING
11:35:26 INFO  State of operation 18168151 : EXECUTING
11:35:28 INFO  State of operation 18168151 : EXECUTING
11:35:31 INFO  State of operation 18168151 : EXECUTING
11:35:34 INFO  State of operation 18168151 : EXECUTING
11:35:36 INFO  State of operation 18168151 : EXECUTING
11:35:38 INFO  State of operation 18168151 : EXECUTING
11:35:40 INFO  State of operation 18168151 : EXECUTING
11:35:43 INFO  State of operation 18168151 : EXECUTING
11:35:45 INFO  State of operation 18168151 : EXECUTING
11:35:47 INFO  State of operation 18168151 : EXECUTING
11:35:49 INFO  State of operation 18168151 : EXECUTING
11:35:52 INFO  State of operation 18168151 : EXECUTING
11:35:54 INFO  State of operation 18168151 : EXECUTING
11:35:56 INFO  State of operation 18168151 : EXECUTING
11:35:59 INFO  State of operation 18168151 : EXECUTING
11:36:01 INFO  State of operation 18168151 : EXECUTING
11:36:04 INFO  State of operation 18168151 : EXECUTING
11:36:06 INFO  State of operation 18168151 : EXECUTING
11:36:08 INFO  State of operation 18168151 : EXECUTING
11:36:11 INFO  State of operation 18168151 : EXECUTING
11:36:13 INFO  State of operation 18168151 : EXECUTING
11:36:16 INFO  State of operation 18168151 : EXECUTING
11:36:18 INFO  State of operation 18168151 : EXECUTING
11:36:21 INFO  State of operation 18168151 : EXECUTING
11:36:23 INFO  State of operation 18168151 : EXECUTING
11:36:26 INFO  State of operation 18168151 : EXECUTING
11:36:28 INFO  State of operation 18168151 : EXECUTING
11:36:31 INFO  State of operation 18168151 : EXECUTING
11:36:33 INFO  State of operation 18168151 : EXECUTING
11:36:35 INFO  State of operation 18168151 : EXECUTING
11:36:37 INFO  State of operation 18168151 : EXECUTING
11:36:39 INFO  State of operation 18168151 : EXECUTING
11:36:42 INFO  State of operation 18168151 : EXECUTING
11:36:44 INFO  State of operation 18168151 : EXECUTING
11:36:46 INFO  State of operation 18168151 : EXECUTING
11:36:49 INFO  State of operation 18168151 : EXECUTING
11:36:52 INFO  State of operation 18168151 : EXECUTING
11:36:54 INFO  State of operation 18168151 : EXECUTING
11:36:57 INFO  State of operation 18168151 : EXECUTING
11:36:59 INFO  State of operation 18168151 : EXECUTING
11:37:01 INFO  State of operation 18168151 : EXECUTING
11:37:03 INFO  State of operation 18168151 : EXECUTING
11:37:06 INFO  State of operation 18168151 : EXECUTING
11:37:08 INFO  State of operation 18168151 : EXECUTING
11:37:10 INFO  State of operation 18168151 : EXECUTING
11:37:13 INFO  State of operation 18168151 : EXECUTING
11:37:15 INFO  State of operation 18168151 : EXECUTING
11:37:18 INFO  State of operation 18168151 : EXECUTING
11:37:20 INFO  State of operation 18168151 : EXECUTING
11:37:23 INFO  State of operation 18168151 : EXECUTING
11:37:25 INFO  State of operation 18168151 : EXECUTING
11:37:28 INFO  State of operation 18168151 : EXECUTING
11:37:30 INFO  State of operation 18168151 : EXECUTING
11:37:33 INFO  State of operation 18168151 : EXECUTING
11:37:35 INFO  State of operation 18168151 : EXECUTING
11:37:38 INFO  State of operation 18168151 : EXECUTING
11:37:40 INFO  State of operation 18168151 : EXECUTING
11:37:42 WARN  SystemError occurred while running test - Timeout while waiting for status SUCCESSFUL or False
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1294, in __call__
    self.testObj.execute()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_remove_multiple_mixed/run.py", line 89, in execute
    self.wait_until_succcess()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 324, in wait_until_succcess
    self.wait_until_status("SUCCESSFUL")
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 381, in wait_until_status
    "Timeout while waiting for status %s or %s" % (status, status2)
SystemError: Timeout while waiting for status SUCCESSFUL or False
11:37:42 WARN  SystemError: Timeout while waiting for status SUCCESSFUL or False ... blocked 
11:37:42 INFO  
11:37:42 INFO  cleanup:
11:37:47 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
11:37:48 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
11:37:48 INFO  
11:37:48 INFO  Test duration: 181.34 secs
11:37:48 INFO  Test final outcome:  BLOCKED
11:37:48 INFO  Test outcome reason: SystemError: Timeout while waiting for status SUCCESSFUL or False (+1 other failures)
11:37:48 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="SoftwareManagement.sm_apt_install_remove_with_version" tests="3" errors="0" failures="0" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_with_version">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it" />
		<system-out>10:34:23 INFO  ==============================================================
10:34:23 INFO  Id:    SoftwareManagement.sm_apt_install_remove_with_version
10:34:23 INFO  Dir:   software_management_end_to_end/
10:34:23 INFO  Title: Validate end to end behaviour for the apt plugin for packages with version
10:34:23 INFO  ==============================================================
10:34:23 INFO  Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it ... skipped
10:34:23 WARN  Aborted test due to SKIPPED outcome
10:34:23 INFO  
10:34:23 INFO  Test duration: 0.02 secs
10:34:23 INFO  Test final outcome:  SKIPPED
10:34:23 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it
10:34:23 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_with_version" />
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_with_version" />
</testsuite><testsuite name="SoftwareManagement.sm_docker_install_remove_multiple" tests="3" errors="0" failures="2" skipped="1" time="0"><testcase classname="SMDockerInstallRemove" name="SoftwareManagement.sm_docker_install_remove_multiple">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it" />
		<system-out>10:34:23 INFO  ==============================================================
10:34:23 INFO  Id:    SoftwareManagement.sm_docker_install_remove_multiple
10:34:23 INFO  Dir:   software_management_end_to_end/
10:34:23 INFO  Title: Validate end to end behaviour for the docker plugin for multiple docker images with mixed versions
10:34:23 INFO  ==============================================================
10:34:23 INFO  Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it ... skipped
10:34:23 WARN  Aborted test due to SKIPPED outcome
10:34:23 INFO  
10:34:23 INFO  Test duration: 0.03 secs
10:34:23 INFO  Test final outcome:  SKIPPED
10:34:23 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it
10:34:23 INFO  
</system-out>
	</testcase>
<testcase classname="SMDockerInstallRemove" name="SoftwareManagement.sm_docker_install_remove_multiple">
		<failure message="BLOCKED: SystemError: The operation has failed" type="BLOCKED" />
		<system-out>11:21:12 INFO  ==============================================================
11:21:12 INFO  Id:    SoftwareManagement.sm_docker_install_remove_multiple
11:21:12 INFO  Dir:   software_management_end_to_end/
11:21:12 INFO  Title: Validate end to end behaviour for the docker plugin for multiple docker images with mixed versions
11:21:12 INFO  ==============================================================
11:21:14 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
11:21:14 INFO  EnvironmentC8y Setup
11:21:15 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
11:21:33 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 17 secs
11:21:33 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
11:21:34 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
11:21:34 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
11:21:36 INFO  State of current operation: SUCCESSFUL
11:21:37 INFO  Response status: 201
11:21:37 INFO  Response to action: {
    "creationTime": "2022-05-02T11:21:37.146Z",
    "deviceName": "m32sd10c",
    "deviceId": "18127529",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18153002",
    "id": "18153002",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '8021526', 'name': 'hello-world', 'url': '', 'version': '::docker'}, {'action': 'install', 'id': '8018911', 'name': 'registry', 'url': '', 'version': '2.6.2::docker'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "hello-world",
            "action": "install",
            "id": "8021526",
            "version": "::docker",
            "url": ""
        },
        {
            "name": "registry",
            "action": "install",
            "id": "8018911",
            "version": "2.6.2::docker",
            "url": ""
        }
    ]
}
11:21:37 INFO  Started operation: {'creationTime': '2022-05-02T11:21:37.146Z', 'deviceName': 'm32sd10c', 'deviceId': '18127529', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18153002', 'id': '18153002', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '8021526', 'name': 'hello-world', 'url': '', 'version': '::docker'}, {'action': 'install', 'id': '8018911', 'name': 'registry', 'url': '', 'version': '2.6.2::docker'}]", 'c8y_SoftwareUpdate': [{'name': 'hello-world', 'action': 'install', 'id': '8021526', 'version': '::docker', 'url': ''}, {'name': 'registry', 'action': 'install', 'id': '8018911', 'version': '2.6.2::docker', 'url': ''}]}
11:21:39 INFO  State of operation 18153002 : PENDING
11:21:41 INFO  State of operation 18153002 : EXECUTING
11:21:43 INFO  State of operation 18153002 : EXECUTING
11:21:46 INFO  State of operation 18153002 : EXECUTING
11:21:48 INFO  State of operation 18153002 : FAILED
11:21:48 ERROR Stopping as the operation has failed
11:21:48 WARN  SystemError occurred while running test - The operation has failed
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1292, in __call__
    self.testObj.setup()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_docker_install_remove_multiple/run.py", line 55, in setup
    self.wait_until_succcess()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 324, in wait_until_succcess
    self.wait_until_status("SUCCESSFUL")
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 361, in wait_until_status
    raise SystemError("The operation has failed")
SystemError: The operation has failed
11:21:48 WARN  SystemError: The operation has failed ... blocked 
11:21:48 INFO  
11:21:48 INFO  cleanup:
11:21:52 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
11:21:53 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
11:21:53 INFO  
11:21:53 INFO  Test duration: 40.45 secs
11:21:53 INFO  Test final outcome:  BLOCKED
11:21:53 INFO  Test outcome reason: SystemError: The operation has failed
11:21:53 INFO  
</system-out>
	</testcase>
<testcase classname="SMDockerInstallRemove" name="SoftwareManagement.sm_docker_install_remove_multiple">
		<failure message="BLOCKED: SystemError: The operation has failed" type="BLOCKED" />
		<system-out>11:40:28 INFO  ==============================================================
11:40:28 INFO  Id:    SoftwareManagement.sm_docker_install_remove_multiple
11:40:28 INFO  Dir:   software_management_end_to_end/
11:40:28 INFO  Title: Validate end to end behaviour for the docker plugin for multiple docker images with mixed versions
11:40:28 INFO  ==============================================================
11:40:28 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
11:40:28 INFO  EnvironmentC8y Setup
11:40:28 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
11:40:28 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 5 seconds
11:40:28 INFO  Delaying execution by 6 seconds
11:40:48 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 13 secs
11:40:49 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
11:40:49 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
11:40:50 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
11:40:52 INFO  State of current operation: SUCCESSFUL
11:40:52 INFO  Response status: 201
11:40:52 INFO  Response to action: {
    "creationTime": "2022-05-02T11:40:52.774Z",
    "deviceName": "m32sd10c",
    "deviceId": "18127529",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18168186",
    "id": "18168186",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '8021526', 'name': 'hello-world', 'url': '', 'version': '::docker'}, {'action': 'install', 'id': '8018911', 'name': 'registry', 'url': '', 'version': '2.6.2::docker'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "hello-world",
            "action": "install",
            "id": "8021526",
            "version": "::docker",
            "url": ""
        },
        {
            "name": "registry",
            "action": "install",
            "id": "8018911",
            "version": "2.6.2::docker",
            "url": ""
        }
    ]
}
11:40:52 INFO  Started operation: {'creationTime': '2022-05-02T11:40:52.774Z', 'deviceName': 'm32sd10c', 'deviceId': '18127529', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18168186', 'id': '18168186', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '8021526', 'name': 'hello-world', 'url': '', 'version': '::docker'}, {'action': 'install', 'id': '8018911', 'name': 'registry', 'url': '', 'version': '2.6.2::docker'}]", 'c8y_SoftwareUpdate': [{'name': 'hello-world', 'action': 'install', 'id': '8021526', 'version': '::docker', 'url': ''}, {'name': 'registry', 'action': 'install', 'id': '8018911', 'version': '2.6.2::docker', 'url': ''}]}
11:40:55 INFO  State of operation 18168186 : PENDING
11:40:57 INFO  State of operation 18168186 : EXECUTING
11:40:59 INFO  State of operation 18168186 : EXECUTING
11:41:01 INFO  State of operation 18168186 : EXECUTING
11:41:03 INFO  State of operation 18168186 : FAILED
11:41:03 ERROR Stopping as the operation has failed
11:41:03 WARN  SystemError occurred while running test - The operation has failed
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1292, in __call__
    self.testObj.setup()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_docker_install_remove_multiple/run.py", line 55, in setup
    self.wait_until_succcess()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 324, in wait_until_succcess
    self.wait_until_status("SUCCESSFUL")
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 361, in wait_until_status
    raise SystemError("The operation has failed")
SystemError: The operation has failed
11:41:03 WARN  SystemError: The operation has failed ... blocked 
11:41:03 INFO  
11:41:03 INFO  cleanup:
11:41:08 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
11:41:08 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
11:41:08 INFO  
11:41:08 INFO  Test duration: 40.42 secs
11:41:08 INFO  Test final outcome:  BLOCKED
11:41:08 INFO  Test outcome reason: SystemError: The operation has failed
11:41:08 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="SoftwareManagement.sm_fake_install_remove" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_fake_install_remove">
		<skipped message="SKIPPED: Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it" />
		<system-out>10:34:23 INFO  ==============================================================
10:34:23 INFO  Id:    SoftwareManagement.sm_fake_install_remove
10:34:23 INFO  Dir:   software_management_end_to_end/
10:34:23 INFO  Title: Validate end to end behaviour for the dummy plugin for installation and removal of packages
10:34:23 INFO  ==============================================================
10:34:23 INFO  Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it ... skipped
10:34:23 WARN  Aborted test due to SKIPPED outcome
10:34:23 INFO  
10:34:23 INFO  Test duration: 0.03 secs
10:34:23 INFO  Test final outcome:  SKIPPED
10:34:23 INFO  Test outcome reason: Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it
10:34:23 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_fake_install_remove" />
</testsuite><testsuite name="SoftwareManagement.sm_fake_install_remove_multiple" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_fake_install_remove_multiple">
		<skipped message="SKIPPED: Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it" />
		<system-out>10:34:23 INFO  ==============================================================
10:34:23 INFO  Id:    SoftwareManagement.sm_fake_install_remove_multiple
10:34:23 INFO  Dir:   software_management_end_to_end/
10:34:23 INFO  Title: Validate end to end behaviour for the dummy-plugin for multiple packages
10:34:23 INFO  ==============================================================
10:34:23 INFO  Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it ... skipped
10:34:23 WARN  Aborted test due to SKIPPED outcome
10:34:23 INFO  
10:34:23 INFO  Test duration: 0.02 secs
10:34:23 INFO  Test final outcome:  SKIPPED
10:34:23 INFO  Test outcome reason: Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it
10:34:23 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_fake_install_remove_multiple" />
</testsuite><testsuite name="SoftwareManagement.sm_mapper_fail_and_reconnect" tests="2" errors="0" failures="0" skipped="2" time="0"><testcase classname="MapperC8yReceiveLastMessageOnRestart" name="SoftwareManagement.sm_mapper_fail_and_reconnect">
		<skipped message="SKIPPED: instable https://github.com/thin-edge/thin-edge.io/issues/752" />
		<system-out>10:34:24 INFO  ==============================================================
10:34:24 INFO  Id:    SoftwareManagement.sm_mapper_fail_and_reconnect
10:34:24 INFO  Dir:   software_management_end_to_end/
10:34:24 INFO  Title: Validate the tedge-mapper-sm-c8y for receiving the last message on restart
10:34:24 INFO  ==============================================================
10:34:24 INFO  instable https://github.com/thin-edge/thin-edge.io/issues/752 ... skipped
10:34:24 INFO  
10:34:24 INFO  Test duration: 0.02 secs
10:34:24 INFO  Test final outcome:  SKIPPED
10:34:24 INFO  Test outcome reason: instable https://github.com/thin-edge/thin-edge.io/issues/752
10:34:24 INFO  
</system-out>
	</testcase>
<testcase classname="MapperC8yReceiveLastMessageOnRestart" name="SoftwareManagement.sm_mapper_fail_and_reconnect">
		<skipped message="SKIPPED: instable https://github.com/thin-edge/thin-edge.io/issues/752" />
		<system-out>11:42:42 INFO  ==============================================================
11:42:42 INFO  Id:    SoftwareManagement.sm_mapper_fail_and_reconnect
11:42:42 INFO  Dir:   software_management_end_to_end/
11:42:42 INFO  Title: Validate the tedge-mapper-sm-c8y for receiving the last message on restart
11:42:42 INFO  ==============================================================
11:42:42 INFO  instable https://github.com/thin-edge/thin-edge.io/issues/752 ... skipped
11:42:42 INFO  
11:42:42 INFO  Test duration: 0.04 secs
11:42:42 INFO  Test final outcome:  SKIPPED
11:42:42 INFO  Test outcome reason: instable https://github.com/thin-edge/thin-edge.io/issues/752
11:42:42 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="SoftwareManagement.sm_mixed_install_remove_multiple" tests="2" errors="0" failures="1" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_mixed_install_remove_multiple">
		<skipped message="SKIPPED: Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it" />
		<system-out>10:34:24 INFO  ==============================================================
10:34:24 INFO  Id:    SoftwareManagement.sm_mixed_install_remove_multiple
10:34:24 INFO  Dir:   software_management_end_to_end/
10:34:24 INFO  Title: Validate end to end behaviour for the dummy-plugin for multiple packages with mixed versions
10:34:24 INFO  ==============================================================
10:34:24 INFO  Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it ... skipped
10:34:24 WARN  Aborted test due to SKIPPED outcome
10:34:24 INFO  
10:34:24 INFO  Test duration: 0.02 secs
10:34:24 INFO  Test final outcome:  SKIPPED
10:34:24 INFO  Test outcome reason: Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it
10:34:24 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_mixed_install_remove_multiple">
		<failure message="BLOCKED: SystemError: The operation has failed (+1 other failures)" type="BLOCKED" />
		<system-out>11:42:42 INFO  ==============================================================
11:42:42 INFO  Id:    SoftwareManagement.sm_mixed_install_remove_multiple
11:42:42 INFO  Dir:   software_management_end_to_end/
11:42:42 INFO  Title: Validate end to end behaviour for the dummy-plugin for multiple packages with mixed versions
11:42:42 INFO  ==============================================================
11:42:42 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
11:42:42 INFO  EnvironmentC8y Setup
11:42:42 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
11:42:42 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 5 seconds
11:42:42 INFO  Delaying execution by 6 seconds
11:43:03 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
11:43:04 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
11:43:05 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
11:43:05 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
11:43:08 INFO  State of current operation: SUCCESSFUL
11:43:08 INFO  Package apple is installed
11:43:08 INFO  Assert that (True == value) with value=True ... passed
11:43:08 INFO  Package banana is installed
11:43:08 INFO  Assert that (True == value) with value=True ... passed
11:43:08 INFO  Package cherry is installed
11:43:08 INFO  Assert that (True == value) with value=True ... passed
11:43:09 INFO  Package asciijump is installed
11:43:09 WARN  Assert that (False == value) with value=True ... failed 
11:43:09 INFO  Assert that (False == value) with value=False ... passed
11:43:09 INFO  Response status: 201
11:43:09 INFO  Response to action: {
    "creationTime": "2022-05-02T11:43:09.625Z",
    "deviceName": "m32sd10c",
    "deviceId": "18127529",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18168253",
    "id": "18168253",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5495053', 'name': 'apple', 'url': ' ', 'version': '::fruits'}, {'action': 'install', 'id': '5494888', 'name': 'banana', 'url': ' ', 'version': '::fruits'}, {'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5495382', 'name': 'cherry', 'url': ' ', 'version': '::fruits'}, {'action': 'install', 'id': '8018911', 'name': 'registry', 'url': ' ', 'version': '::docker'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "apple",
            "action": "install",
            "id": "5495053",
            "version": "::fruits",
            "url": " "
        },
        {
            "name": "banana",
            "action": "install",
            "id": "5494888",
            "version": "::fruits",
            "url": " "
        },
        {
            "name": "asciijump",
            "action": "install",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "cherry",
            "action": "install",
            "id": "5495382",
            "version": "::fruits",
            "url": " "
        },
        {
            "name": "registry",
            "action": "install",
            "id": "8018911",
            "version": "::docker",
            "url": " "
        }
    ]
}
11:43:09 INFO  Started operation: {'creationTime': '2022-05-02T11:43:09.625Z', 'deviceName': 'm32sd10c', 'deviceId': '18127529', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18168253', 'id': '18168253', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5495053', 'name': 'apple', 'url': ' ', 'version': '::fruits'}, {'action': 'install', 'id': '5494888', 'name': 'banana', 'url': ' ', 'version': '::fruits'}, {'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5495382', 'name': 'cherry', 'url': ' ', 'version': '::fruits'}, {'action': 'install', 'id': '8018911', 'name': 'registry', 'url': ' ', 'version': '::docker'}]", 'c8y_SoftwareUpdate': [{'name': 'apple', 'action': 'install', 'id': '5495053', 'version': '::fruits', 'url': ' '}, {'name': 'banana', 'action': 'install', 'id': '5494888', 'version': '::fruits', 'url': ' '}, {'name': 'asciijump', 'action': 'install', 'id': '5475369', 'version': '::apt', 'url': ' '}, {'name': 'cherry', 'action': 'install', 'id': '5495382', 'version': '::fruits', 'url': ' '}, {'name': 'registry', 'action': 'install', 'id': '8018911', 'version': '::docker', 'url': ' '}]}
11:43:11 INFO  State of operation 18168253 : PENDING
11:43:14 INFO  State of operation 18168253 : EXECUTING
11:43:16 INFO  State of operation 18168253 : EXECUTING
11:43:18 INFO  State of operation 18168253 : EXECUTING
11:43:20 INFO  State of operation 18168253 : EXECUTING
11:43:22 INFO  State of operation 18168253 : EXECUTING
11:43:25 INFO  State of operation 18168253 : EXECUTING
11:43:29 INFO  State of operation 18168253 : EXECUTING
11:43:31 INFO  State of operation 18168253 : EXECUTING
11:43:33 INFO  State of operation 18168253 : EXECUTING
11:43:36 INFO  State of operation 18168253 : EXECUTING
11:43:38 INFO  State of operation 18168253 : EXECUTING
11:43:40 INFO  State of operation 18168253 : EXECUTING
11:43:42 INFO  State of operation 18168253 : EXECUTING
11:43:44 INFO  State of operation 18168253 : EXECUTING
11:43:47 INFO  State of operation 18168253 : EXECUTING
11:43:49 INFO  State of operation 18168253 : EXECUTING
11:43:51 INFO  State of operation 18168253 : FAILED
11:43:51 ERROR Stopping as the operation has failed
11:43:51 WARN  SystemError occurred while running test - The operation has failed
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1294, in __call__
    self.testObj.execute()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_mixed_install_remove_multiple/run.py", line 95, in execute
    self.wait_until_succcess()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 324, in wait_until_succcess
    self.wait_until_status("SUCCESSFUL")
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 361, in wait_until_status
    raise SystemError("The operation has failed")
SystemError: The operation has failed
11:43:51 WARN  SystemError: The operation has failed ... blocked 
11:43:51 INFO  
11:43:51 INFO  cleanup:
11:43:55 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
11:43:56 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
11:43:56 INFO  
11:43:56 INFO  Test duration: 73.83 secs
11:43:56 INFO  Test final outcome:  BLOCKED
11:43:56 INFO  Test outcome reason: SystemError: The operation has failed (+1 other failures)
11:43:56 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="agent_init_session" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="AgentInitSession" name="agent_init_session">
		<failure message="FAILED: Grep on sub_resp.out contains &quot;Ld3KgqpcLDlrYH6sfpG7w&quot;" type="FAILED" />
		<system-out>10:35:17 INFO  ==============================================================
10:35:17 INFO  Id:    agent_init_session
10:35:17 INFO  Dir:   tedge_agent/
10:35:17 INFO  Title: Validate tedge agent init session feature
10:35:18 INFO  ==============================================================
10:35:18 INFO  Executed sudo&lt;remove_lock&gt;, exit status 0
10:35:19 INFO  Executed sudo&lt;agent_clear&gt;, exit status 0
10:35:19 INFO  Executed sudo&lt;agent_init&gt;, exit status 0
10:35:20 INFO  Executed sudo&lt;pub_req&gt;, exit status 0
10:35:20 INFO  Started sudo&lt;sub_resp&gt; with process id 28647
10:35:20 INFO  Executed sudo&lt;agent_start&gt;, exit status 0
10:35:20 INFO  Waiting for 2.0 seconds
10:35:23 INFO  Executed sudo&lt;agent_stop&gt;, exit status 0
10:35:23 INFO  Executed sudo&lt;sub_stop&gt;, exit status 0
10:35:23 WARN  Grep on sub_resp.out contains "Ld3KgqpcLDlrYH6sfpG7w" ... failed [run.py:87]
10:35:23 INFO  
10:35:23 INFO  Test duration: 5.88 secs
10:35:23 INFO  Test final outcome:  FAILED
10:35:23 INFO  Test outcome reason: Grep on sub_resp.out contains "Ld3KgqpcLDlrYH6sfpG7w"
10:35:23 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="apt_fail_noarg" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginFailNoArg" name="apt_fail_noarg">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>10:24:42 INFO  ==============================================================
10:24:42 INFO  Id:    apt_fail_noarg
10:24:42 INFO  Dir:   plugin_apt/
10:24:42 INFO  Title: Validate apt plugin fetures(prepare, list, install, remove finalize) without arguments
10:24:42 INFO  ==============================================================
10:24:42 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
10:24:42 WARN  Aborted test due to SKIPPED outcome
10:24:42 INFO  
10:24:42 INFO  Test duration: 0.04 secs
10:24:42 INFO  Test final outcome:  SKIPPED
10:24:42 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
10:24:42 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginFailNoArg" name="apt_fail_noarg" />
</testsuite><testsuite name="apt_finalize" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginFinalize" name="apt_finalize">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>10:24:42 INFO  ==============================================================
10:24:42 INFO  Id:    apt_finalize
10:24:42 INFO  Dir:   plugin_apt/
10:24:42 INFO  Title: Validate apt plugin finalize - dense version
10:24:42 INFO  ==============================================================
10:24:42 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
10:24:42 WARN  Aborted test due to SKIPPED outcome
10:24:42 INFO  
10:24:42 INFO  Test duration: 0.03 secs
10:24:42 INFO  Test final outcome:  SKIPPED
10:24:42 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
10:24:42 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginFinalize" name="apt_finalize" />
</testsuite><testsuite name="apt_install" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginInstallDense" name="apt_install">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>10:24:42 INFO  ==============================================================
10:24:42 INFO  Id:    apt_install
10:24:42 INFO  Dir:   plugin_apt/
10:24:42 INFO  Title: Validate apt plugin prepare - dense version
10:24:42 INFO  ==============================================================
10:24:42 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
10:24:42 WARN  Aborted test due to SKIPPED outcome
10:24:42 INFO  
10:24:42 INFO  Test duration: 0.02 secs
10:24:42 INFO  Test final outcome:  SKIPPED
10:24:42 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
10:24:42 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginInstallDense" name="apt_install" />
</testsuite><testsuite name="apt_install_fail" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginInstallFail" name="apt_install_fail">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>10:24:42 INFO  ==============================================================
10:24:42 INFO  Id:    apt_install_fail
10:24:42 INFO  Dir:   plugin_apt/
10:24:42 INFO  Title: Validate apt plugin prepare - failing
10:24:42 INFO  ==============================================================
10:24:42 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
10:24:42 WARN  Aborted test due to SKIPPED outcome
10:24:42 INFO  
10:24:42 INFO  Test duration: 0.02 secs
10:24:42 INFO  Test final outcome:  SKIPPED
10:24:42 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
10:24:42 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginInstallFail" name="apt_install_fail" />
</testsuite><testsuite name="apt_install_local_file" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginInstallFromLocalFile" name="apt_install_local_file">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>10:24:42 INFO  ==============================================================
10:24:42 INFO  Id:    apt_install_local_file
10:24:42 INFO  Dir:   plugin_apt/
10:24:42 INFO  Title: Validate apt plugin install from local file - SUCCESS case
10:24:42 INFO  ==============================================================
10:24:42 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
10:24:42 WARN  Aborted test due to SKIPPED outcome
10:24:42 INFO  
10:24:42 INFO  Test duration: 0.02 secs
10:24:42 INFO  Test final outcome:  SKIPPED
10:24:42 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
10:24:42 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginInstallFromLocalFile" name="apt_install_local_file" />
</testsuite><testsuite name="apt_install_local_file_fail" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginInstallFromLocalFileFail" name="apt_install_local_file_fail">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>10:24:42 INFO  ==============================================================
10:24:42 INFO  Id:    apt_install_local_file_fail
10:24:42 INFO  Dir:   plugin_apt/
10:24:42 INFO  Title: Validate apt plugin install from local file - FAIL case
10:24:42 INFO  ==============================================================
10:24:42 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
10:24:42 WARN  Aborted test due to SKIPPED outcome
10:24:42 INFO  
10:24:42 INFO  Test duration: 0.02 secs
10:24:42 INFO  Test final outcome:  SKIPPED
10:24:42 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
10:24:42 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginInstallFromLocalFileFail" name="apt_install_local_file_fail" />
</testsuite><testsuite name="apt_install_local_file_with_version" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginInstallFromLocalFileWithVersion" name="apt_install_local_file_with_version">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>10:24:42 INFO  ==============================================================
10:24:42 INFO  Id:    apt_install_local_file_with_version
10:24:42 INFO  Dir:   plugin_apt/
10:24:42 INFO  Title: Validate apt plugin install from local file AND with a version - SUCCESS case
10:24:42 INFO  ==============================================================
10:24:42 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
10:24:42 WARN  Aborted test due to SKIPPED outcome
10:24:42 INFO  
10:24:42 INFO  Test duration: 0.03 secs
10:24:42 INFO  Test final outcome:  SKIPPED
10:24:42 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
10:24:42 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginInstallFromLocalFileWithVersion" name="apt_install_local_file_with_version" />
</testsuite><testsuite name="apt_install_local_file_with_version_fail" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginInstallFromLocalFileWithVersionFail" name="apt_install_local_file_with_version_fail">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>10:24:42 INFO  ==============================================================
10:24:42 INFO  Id:    apt_install_local_file_with_version_fail
10:24:42 INFO  Dir:   plugin_apt/
10:24:42 INFO  Title: Validate apt plugin install from local file AND with a version - FAIL case
10:24:42 INFO  ==============================================================
10:24:42 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
10:24:42 WARN  Aborted test due to SKIPPED outcome
10:24:42 INFO  
10:24:42 INFO  Test duration: 0.03 secs
10:24:42 INFO  Test final outcome:  SKIPPED
10:24:42 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
10:24:42 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginInstallFromLocalFileWithVersionFail" name="apt_install_local_file_with_version_fail" />
</testsuite><testsuite name="apt_list_format" tests="2" errors="0" failures="0" skipped="0" time="0"><testcase classname="AptPluginListFormat" name="apt_list_format" />
<testcase classname="AptPluginListFormat" name="apt_list_format" />
</testsuite><testsuite name="apt_prepare" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginPrepare" name="apt_prepare">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>10:24:45 INFO  ==============================================================
10:24:45 INFO  Id:    apt_prepare
10:24:45 INFO  Dir:   plugin_apt/
10:24:45 INFO  Title: Validate apt plugin install - dense version
10:24:45 INFO  ==============================================================
10:24:45 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
10:24:45 WARN  Aborted test due to SKIPPED outcome
10:24:45 INFO  
10:24:45 INFO  Test duration: 0.02 secs
10:24:45 INFO  Test final outcome:  SKIPPED
10:24:45 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
10:24:45 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginPrepare" name="apt_prepare" />
</testsuite><testsuite name="apt_prepare_install_finalize" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginPrepareInstallFinalize" name="apt_prepare_install_finalize">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>10:24:45 INFO  ==============================================================
10:24:45 INFO  Id:    apt_prepare_install_finalize
10:24:45 INFO  Dir:   plugin_apt/
10:24:45 INFO  Title: Validate apt plugin install use case
10:24:45 INFO  ==============================================================
10:24:45 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
10:24:45 WARN  Aborted test due to SKIPPED outcome
10:24:45 INFO  
10:24:45 INFO  Test duration: 0.02 secs
10:24:45 INFO  Test final outcome:  SKIPPED
10:24:45 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
10:24:45 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginPrepareInstallFinalize" name="apt_prepare_install_finalize" />
</testsuite><testsuite name="apt_remove" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginRemoveDense" name="apt_remove">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>10:24:45 INFO  ==============================================================
10:24:45 INFO  Id:    apt_remove
10:24:45 INFO  Dir:   plugin_apt/
10:24:45 INFO  Title: Validate apt plugin prepare - dense version
10:24:45 INFO  ==============================================================
10:24:45 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
10:24:45 WARN  Aborted test due to SKIPPED outcome
10:24:45 INFO  
10:24:45 INFO  Test duration: 0.02 secs
10:24:45 INFO  Test final outcome:  SKIPPED
10:24:45 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
10:24:45 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginRemoveDense" name="apt_remove" />
</testsuite><testsuite name="apt_remove_fail" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginRemoveFails" name="apt_remove_fail">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>10:24:45 INFO  ==============================================================
10:24:45 INFO  Id:    apt_remove_fail
10:24:45 INFO  Dir:   plugin_apt/
10:24:45 INFO  Title: Validate apt plugin prepare - failing
10:24:45 INFO  ==============================================================
10:24:45 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
10:24:45 WARN  Aborted test due to SKIPPED outcome
10:24:45 INFO  
10:24:45 INFO  Test duration: 0.02 secs
10:24:45 INFO  Test final outcome:  SKIPPED
10:24:45 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
10:24:45 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginRemoveFails" name="apt_remove_fail" />
</testsuite><testsuite name="apt_sequence_install_with_version" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginPrepInstallWithVersionFinalize" name="apt_sequence_install_with_version">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>10:24:45 INFO  ==============================================================
10:24:45 INFO  Id:    apt_sequence_install_with_version
10:24:45 INFO  Dir:   plugin_apt/
10:24:45 INFO  Title: Validate apt plugin install with version use case
10:24:45 INFO  ==============================================================
10:24:45 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
10:24:45 WARN  Aborted test due to SKIPPED outcome
10:24:45 INFO  
10:24:45 INFO  Test duration: 0.02 secs
10:24:45 INFO  Test final outcome:  SKIPPED
10:24:45 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
10:24:45 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginPrepInstallWithVersionFinalize" name="apt_sequence_install_with_version" />
</testsuite><testsuite name="apt_sequence_install_with_version_fail" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginInstallWithVersionFails" name="apt_sequence_install_with_version_fail">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>10:24:46 INFO  ==============================================================
10:24:46 INFO  Id:    apt_sequence_install_with_version_fail
10:24:46 INFO  Dir:   plugin_apt/
10:24:46 INFO  Title: Validate apt plugin install with version fails use case
10:24:46 INFO  ==============================================================
10:24:46 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
10:24:46 WARN  Aborted test due to SKIPPED outcome
10:24:46 INFO  
10:24:46 INFO  Test duration: 0.02 secs
10:24:46 INFO  Test final outcome:  SKIPPED
10:24:46 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
10:24:46 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginInstallWithVersionFails" name="apt_sequence_install_with_version_fail" />
</testsuite><testsuite name="apt_sequence_remove" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginInstallPrepareRemoveFinalize" name="apt_sequence_remove">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>10:24:46 INFO  ==============================================================
10:24:46 INFO  Id:    apt_sequence_remove
10:24:46 INFO  Dir:   plugin_apt/
10:24:46 INFO  Title: Validate apt plugin prepare install remove finalize use case
10:24:46 INFO  ==============================================================
10:24:46 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
10:24:46 WARN  Aborted test due to SKIPPED outcome
10:24:46 INFO  
10:24:46 INFO  Test duration: 0.02 secs
10:24:46 INFO  Test final outcome:  SKIPPED
10:24:46 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
10:24:46 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginInstallPrepareRemoveFinalize" name="apt_sequence_remove" />
</testsuite><testsuite name="apt_sequence_remove_with_version" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginRemoveWithVersion" name="apt_sequence_remove_with_version">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>10:24:46 INFO  ==============================================================
10:24:46 INFO  Id:    apt_sequence_remove_with_version
10:24:46 INFO  Dir:   plugin_apt/
10:24:46 INFO  Title: Validate apt plugin remove with version use case
10:24:46 INFO  ==============================================================
10:24:46 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
10:24:46 WARN  Aborted test due to SKIPPED outcome
10:24:46 INFO  
10:24:46 INFO  Test duration: 0.02 secs
10:24:46 INFO  Test final outcome:  SKIPPED
10:24:46 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
10:24:46 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginRemoveWithVersion" name="apt_sequence_remove_with_version" />
</testsuite><testsuite name="apt_sequence_remove_with_version_fail" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginRemoveWithVersionFails" name="apt_sequence_remove_with_version_fail">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>10:24:46 INFO  ==============================================================
10:24:46 INFO  Id:    apt_sequence_remove_with_version_fail
10:24:46 INFO  Dir:   plugin_apt/
10:24:46 INFO  Title: Validate apt plugin remove with version fails when the version not found
10:24:46 INFO  ==============================================================
10:24:46 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
10:24:46 WARN  Aborted test due to SKIPPED outcome
10:24:46 INFO  
10:24:46 INFO  Test duration: 0.02 secs
10:24:46 INFO  Test final outcome:  SKIPPED
10:24:46 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
10:24:46 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginRemoveWithVersionFails" name="apt_sequence_remove_with_version_fail" />
</testsuite><testsuite name="apt_update_list" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginUpdateList" name="apt_update_list">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>10:24:46 INFO  ==============================================================
10:24:46 INFO  Id:    apt_update_list
10:24:46 INFO  Dir:   plugin_apt/
10:24:46 INFO  Title: Validate apt plugin update-list command use case
10:24:46 INFO  ==============================================================
10:24:46 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
10:24:46 WARN  Aborted test due to SKIPPED outcome
10:24:46 INFO  
10:24:46 INFO  Test duration: 0.02 secs
10:24:46 INFO  Test final outcome:  SKIPPED
10:24:46 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
10:24:46 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginUpdateList" name="apt_update_list" />
</testsuite><testsuite name="apt_wrong_option" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginWrongOption" name="apt_wrong_option">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>10:24:46 INFO  ==============================================================
10:24:46 INFO  Id:    apt_wrong_option
10:24:46 INFO  Dir:   plugin_apt/
10:24:46 INFO  Title: Validate apt plugin install
10:24:46 INFO  ==============================================================
10:24:46 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
10:24:46 WARN  Aborted test due to SKIPPED outcome
10:24:46 INFO  
10:24:46 INFO  Test duration: 0.02 secs
10:24:46 INFO  Test final outcome:  SKIPPED
10:24:46 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
10:24:46 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginWrongOption" name="apt_wrong_option" />
</testsuite><testsuite name="c8y_jwt_token_retrieval" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="ValidateJWTTokenRetrieval" name="c8y_jwt_token_retrieval">
		<failure message="FAILED: Grep on resp_jwt.out contains &quot;\[c8y/s/dat\] 71,&quot;" type="FAILED" />
		<system-out>10:21:02 INFO  ==============================================================
10:21:02 INFO  Id:    c8y_jwt_token_retrieval
10:21:02 INFO  Dir:   cloud_c8y/
10:21:02 INFO  Title: Validate the Cumulocity JWT Token Retrieval feature
10:21:02 INFO  ==============================================================
10:21:02 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
10:21:18 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 15 secs
10:21:18 INFO  Started sudo&lt;resp_jwt&gt; with process id 24621
10:21:18 INFO  Executed sudo&lt;req_jwt&gt;, exit status 0
10:21:20 INFO  Executed sudo&lt;kill_out&gt;, exit status 0
10:21:20 WARN  Grep on resp_jwt.out contains "\[c8y/s/dat\] 71," ... failed [run.py:60]
10:21:20 INFO  
10:21:20 INFO  cleanup:
10:21:24 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
10:21:24 INFO  
10:21:24 INFO  Test duration: 22.08 secs
10:21:24 INFO  Test final outcome:  FAILED
10:21:24 INFO  Test outcome reason: Grep on resp_jwt.out contains "\[c8y/s/dat\] 71,"
10:21:24 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="c8y_restart_bridge" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="RestartBridge" name="c8y_restart_bridge" />
</testsuite><testsuite name="call_tedge_-V" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="call_tedge_-V" />
</testsuite><testsuite name="call_tedge_-h" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="call_tedge_-h" />
</testsuite><testsuite name="call_tedge_-h_-V" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="call_tedge_-h_-V" />
</testsuite><testsuite name="call_tedge_config_help" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="call_tedge_config_help" />
</testsuite><testsuite name="call_tedge_config_list" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="call_tedge_config_list" />
</testsuite><testsuite name="custom_init_system" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="CustomInitSystem" name="custom_init_system" />
</testsuite><testsuite name="docker_plugin_finalize" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="DockerPluginFinalizeTest" name="docker_plugin_finalize">
		<skipped message="SKIPPED: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'" />
		<system-out>10:24:46 INFO  ==============================================================
10:24:46 INFO  Id:    docker_plugin_finalize
10:24:46 INFO  Dir:   plugin_docker/
10:24:46 INFO  Title: Validate docker plugin finalize command
10:24:46 INFO  ==============================================================
10:24:46 INFO  Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container' ... skipped
10:24:46 WARN  Aborted test due to SKIPPED outcome
10:24:46 INFO  
10:24:46 INFO  Test duration: 0.04 secs
10:24:46 INFO  Test final outcome:  SKIPPED
10:24:46 INFO  Test outcome reason: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'
10:24:46 INFO  
</system-out>
	</testcase>
<testcase classname="DockerPluginFinalizeTest" name="docker_plugin_finalize" />
</testsuite><testsuite name="docker_plugin_install" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="DockerPluginInstallTest" name="docker_plugin_install">
		<skipped message="SKIPPED: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'" />
		<system-out>10:24:46 INFO  ==============================================================
10:24:46 INFO  Id:    docker_plugin_install
10:24:46 INFO  Dir:   plugin_docker/
10:24:46 INFO  Title: Validate docker plugin install
10:24:46 INFO  ==============================================================
10:24:46 INFO  Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container' ... skipped
10:24:46 WARN  Aborted test due to SKIPPED outcome
10:24:46 INFO  
10:24:46 INFO  Test duration: 0.02 secs
10:24:46 INFO  Test final outcome:  SKIPPED
10:24:46 INFO  Test outcome reason: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'
10:24:46 INFO  
</system-out>
	</testcase>
<testcase classname="DockerPluginInstallTest" name="docker_plugin_install" />
</testsuite><testsuite name="docker_plugin_install_with_version" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="DockerPluginInstallWithVersionTest" name="docker_plugin_install_with_version">
		<skipped message="SKIPPED: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'" />
		<system-out>10:24:46 INFO  ==============================================================
10:24:46 INFO  Id:    docker_plugin_install_with_version
10:24:46 INFO  Dir:   plugin_docker/
10:24:46 INFO  Title: Validate docker plugin install
10:24:46 INFO  ==============================================================
10:24:46 INFO  Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container' ... skipped
10:24:46 WARN  Aborted test due to SKIPPED outcome
10:24:46 INFO  
10:24:46 INFO  Test duration: 0.02 secs
10:24:46 INFO  Test final outcome:  SKIPPED
10:24:46 INFO  Test outcome reason: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'
10:24:46 INFO  
</system-out>
	</testcase>
<testcase classname="DockerPluginInstallWithVersionTest" name="docker_plugin_install_with_version" />
</testsuite><testsuite name="docker_plugin_remove" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="DockerPluginRemoveTest" name="docker_plugin_remove">
		<skipped message="SKIPPED: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'" />
		<system-out>10:24:46 INFO  ==============================================================
10:24:46 INFO  Id:    docker_plugin_remove
10:24:46 INFO  Dir:   plugin_docker/
10:24:46 INFO  Title: Validate docker plugin remove
10:24:46 INFO  ==============================================================
10:24:46 INFO  Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container' ... skipped
10:24:46 WARN  Aborted test due to SKIPPED outcome
10:24:46 INFO  
10:24:46 INFO  Test duration: 0.02 secs
10:24:46 INFO  Test final outcome:  SKIPPED
10:24:46 INFO  Test outcome reason: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'
10:24:46 INFO  
</system-out>
	</testcase>
<testcase classname="DockerPluginRemoveTest" name="docker_plugin_remove" />
</testsuite><testsuite name="docker_plugin_remove_with_version" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="DockerPluginRemoveWithVersionTest" name="docker_plugin_remove_with_version">
		<skipped message="SKIPPED: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'" />
		<system-out>10:24:46 INFO  ==============================================================
10:24:46 INFO  Id:    docker_plugin_remove_with_version
10:24:46 INFO  Dir:   plugin_docker/
10:24:46 INFO  Title: Validate docker plugin remove with version
10:24:46 INFO  ==============================================================
10:24:46 INFO  Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container' ... skipped
10:24:46 WARN  Aborted test due to SKIPPED outcome
10:24:46 INFO  
10:24:46 INFO  Test duration: 0.02 secs
10:24:46 INFO  Test final outcome:  SKIPPED
10:24:46 INFO  Test outcome reason: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'
10:24:46 INFO  
</system-out>
	</testcase>
<testcase classname="DockerPluginRemoveWithVersionTest" name="docker_plugin_remove_with_version" />
</testsuite><testsuite name="docker_plugin_update" tests="2" errors="0" failures="1" skipped="1" time="0"><testcase classname="DockerPluginUpdateTest" name="docker_plugin_update">
		<skipped message="SKIPPED: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'" />
		<system-out>10:24:46 INFO  ==============================================================
10:24:46 INFO  Id:    docker_plugin_update
10:24:46 INFO  Dir:   plugin_docker/
10:24:46 INFO  Title: Validate docker plugin install updates existing containers
10:24:46 INFO  ==============================================================
10:24:46 INFO  Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container' ... skipped
10:24:46 WARN  Aborted test due to SKIPPED outcome
10:24:46 INFO  
10:24:46 INFO  Test duration: 0.02 secs
10:24:46 INFO  Test final outcome:  SKIPPED
10:24:46 INFO  Test outcome reason: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'
10:24:46 INFO  
</system-out>
	</testcase>
<testcase classname="DockerPluginUpdateTest" name="docker_plugin_update">
		<failure message="BLOCKED: sudo&lt;plugin_install&gt; returned non-zero exit code 2 (+1 other failures)" type="BLOCKED" />
		<system-out>11:20:34 INFO  ==============================================================
11:20:34 INFO  Id:    docker_plugin_update
11:20:34 INFO  Dir:   plugin_docker/
11:20:34 INFO  Title: Validate docker plugin install updates existing containers
11:20:34 INFO  ==============================================================
11:20:35 INFO  Executed sudo&lt;docker_images&gt;, exit status 0
11:20:35 INFO  Grep on file /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_update/Output/linux/docker_images.out ... passed
11:20:58 INFO  Executed sudo&lt;docker_run&gt;, exit status 0, duration 23 secs
11:20:59 INFO  Executed sudo&lt;docker_ps&gt;, exit status 0
11:20:59 INFO  Grep on file /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_update/Output/linux/docker_ps.out ... passed
11:21:05 WARN  Executed sudo&lt;plugin_install&gt;, exit status 2
11:21:05 WARN  sudo&lt;plugin_install&gt; returned non-zero exit code 2 ... blocked [environment_docker_plugin.py:136,environment_docker_plugin.py:144,run.py:30]
11:21:06 INFO  Executed sudo&lt;docker_ps_latest&gt;, exit status 0
11:21:07 INFO  Executed sudo&lt;docker_ps&gt;, exit status 0
11:21:07 INFO  Grep on file /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_update/Output/linux/docker_ps.out ... passed
11:21:08 INFO  Executed sudo&lt;docker_ps&gt;, exit status 0
11:21:08 WARN  Grep on /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_update/Output/linux/docker_ps.out contains "registry:2.7.1" ... failed [environment_docker_plugin.py:49,run.py:39]
11:21:08 INFO  
11:21:08 INFO  cleanup:
11:21:08 INFO  Removing containers scheduled to be cleaned up: ['f133209a1b27', 'be91eb581d16', '20f1395c1f1da9ffb2043b8087d5ddf6b169d7f279a3ed237894d27b7d223fa0\n', '5f769f93ebf60b2ee225f9ff4379a2f2069c75c617d3fe918db75e95a6be61d4\n', '4fa45b80cee608d793aa60cf5430f576c6370841c5bee5a3ed3d3d2cebed424c\n', '']
11:21:09 WARN  Executed sudo&lt;containers_for_cleanup&gt;, exit status 1
11:21:09 INFO  Removing images scheduled to be cleaned up: ['registry', 'registry', 'registry:2.7.1', 'registry', 'registry:2.7.1', 'registry:2.6.2', 'registry:2.7.1']
11:21:12 INFO  Executed sudo&lt;containers_for_cleanup&gt;, exit status 0
11:21:12 INFO  
11:21:12 INFO  Test duration: 38.41 secs
11:21:12 INFO  Test final outcome:  BLOCKED
11:21:12 INFO  Test outcome reason: sudo&lt;plugin_install&gt; returned non-zero exit code 2 (+1 other failures)
11:21:12 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="invalid_device_id" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="InvalidDeviceId" name="invalid_device_id" />
</testsuite><testsuite name="local_mqtt_pub_fail_noqos" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="local_mqtt_pub_fail_noqos" />
</testsuite><testsuite name="local_mqtt_pub_fail_qos0" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="local_mqtt_pub_fail_qos0" />
</testsuite><testsuite name="local_mqtt_pub_fail_qos1" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="local_mqtt_pub_fail_qos1" />
</testsuite><testsuite name="local_mqtt_pub_fail_qos2" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="local_mqtt_pub_fail_qos2" />
</testsuite><testsuite name="local_mqtt_pub_sub" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="local_mqtt_pub_sub" />
</testsuite><testsuite name="local_mqtt_pub_sub_retain" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="MqttPublishWithRetain" name="local_mqtt_pub_sub_retain" />
</testsuite><testsuite name="local_mqtt_pub_sub_with_sudo" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="local_mqtt_pub_sub_with_sudo" />
</testsuite><testsuite name="local_mqtt_sub_fail" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="local_mqtt_sub_fail" />
</testsuite><testsuite name="log_request_error_messages" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="LogRequestVerifySearchTextError" name="log_request_error_messages">
		<failure message="FAILED: Assert that (True == value) with value=False" type="FAILED" />
		<system-out>10:21:43 INFO  ==============================================================
10:21:43 INFO  Id:    log_request_error_messages
10:21:43 INFO  Dir:   log_request_end_to_end/
10:21:43 INFO  Title: Validate end to end behaviour for the log request operation check for error messages
10:21:43 INFO  ==============================================================
10:21:45 INFO  EnvironmentC8y Setup
10:21:45 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
10:21:45 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 2 seconds
10:21:45 INFO  Delaying execution by 9 seconds
10:22:08 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
10:22:09 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
10:22:10 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
10:22:10 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
10:22:11 INFO  Executed sudo&lt;rm_logs&gt;, exit status 0
10:22:11 INFO  Executed sudo&lt;move_logs&gt;, exit status 0
10:22:12 INFO  log response None
10:22:12 WARN  retry, retrying in 1 seconds...
10:22:13 INFO  log response https://thin-edge-io.eu-latest.cumulocity.com/event/events/18128871/binaries
10:22:13 WARN  Assert that (True == value) with value=False ... failed [run.py:43]
10:22:13 INFO  
10:22:13 INFO  cleanup:
10:22:14 INFO  Executed sudo&lt;rm_logs&gt;, exit status 0
10:22:14 INFO  Executed sudo&lt;rm_tmp_logs&gt;, exit status 0
10:22:15 INFO  Executed sudo&lt;send_failed&gt;, exit status 0
10:22:19 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
10:22:20 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
10:22:20 INFO  
10:22:20 INFO  Test duration: 36.22 secs
10:22:20 INFO  Test final outcome:  FAILED
10:22:20 INFO  Test outcome reason: Assert that (True == value) with value=False
10:22:20 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="log_request_num_lines" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="LogRequestVerifyNumberOfLines" name="log_request_num_lines">
		<failure message="FAILED: Assert that (True == value) with value=False" type="FAILED" />
		<system-out>10:22:20 INFO  ==============================================================
10:22:20 INFO  Id:    log_request_num_lines
10:22:20 INFO  Dir:   log_request_end_to_end/
10:22:20 INFO  Title: Validate end to end behaviour for the log request operation check for number of lines
10:22:20 INFO  ==============================================================
10:22:20 INFO  EnvironmentC8y Setup
10:22:20 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
10:22:20 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 5 seconds
10:22:20 INFO  Delaying execution by 6 seconds
10:22:41 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
10:22:41 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
10:22:42 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
10:22:42 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
10:22:43 INFO  Executed sudo&lt;rm_logs&gt;, exit status 0
10:22:43 INFO  Executed sudo&lt;move_logs&gt;, exit status 0
10:22:44 INFO  log response None
10:22:44 WARN  retry, retrying in 1 seconds...
10:22:45 INFO  log response https://thin-edge-io.eu-latest.cumulocity.com/event/events/18129272/binaries
10:22:45 INFO  nlines 0
10:22:45 WARN  Assert that (True == value) with value=False ... failed [run.py:44]
10:22:45 INFO  
10:22:45 INFO  cleanup:
10:22:46 INFO  Executed sudo&lt;rm_logs&gt;, exit status 0
10:22:46 INFO  Executed sudo&lt;rm_tmp_logs&gt;, exit status 0
10:22:47 INFO  Executed sudo&lt;send_failed&gt;, exit status 0
10:22:51 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
10:22:52 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
10:22:52 INFO  
10:22:52 INFO  Test duration: 31.95 secs
10:22:52 INFO  Test final outcome:  FAILED
10:22:52 INFO  Test outcome reason: Assert that (True == value) with value=False
10:22:52 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="mapper_awaits_before_reconnect" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="MapperReconnectAwait" name="mapper_awaits_before_reconnect">
		<failure message="FAILED: Line count on tedge_mapper_journal.out for &quot;tedge_mapper::core::mapper: MQTT connection error:&quot; expected &gt;=2 but got 0" type="FAILED" />
		<system-out>10:37:34 INFO  ==============================================================
10:37:34 INFO  Id:    mapper_awaits_before_reconnect
10:37:34 INFO  Dir:   tedge_mapper_c8y/
10:37:34 INFO  Title: Validate mapper doesn't reconnect too often
10:37:34 INFO  ==============================================================
10:37:34 INFO  Started mosquitto_sub with process id 29402
10:37:34 INFO  Waiting for 0.1 seconds
10:37:35 INFO  Executed sudo&lt;tedge_mapper&gt;, exit status 0
10:37:35 INFO  Waiting for 5.0 seconds
10:37:40 INFO  Executed sudo&lt;tedge_mapper&gt;, exit status 0
10:37:41 INFO  Executed sudo&lt;tedge_mapper_journal&gt;, exit status 0
10:37:41 WARN  Grep on tedge_mapper_journal.out contains "ERROR" ... failed [run.py:67]
10:37:41 INFO  Line count on file tedge_mapper_journal.out ... passed
10:37:41 WARN  Line count on tedge_mapper_journal.out for "tedge_mapper::core::mapper: MQTT connection error:" expected &gt;=2 but got 0 ... failed [run.py:78]
10:37:41 WARN  Aborted test due to FAILED outcome
10:37:41 INFO  
10:37:41 INFO  Test duration: 6.97 secs
10:37:41 INFO  Test final outcome:  FAILED
10:37:41 INFO  Test outcome reason: Line count on tedge_mapper_journal.out for "tedge_mapper::core::mapper: MQTT connection error:" expected &gt;=2 but got 0
10:37:41 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="monitoring_smaller_interval" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="MonitoringSmallInterval" name="monitoring_smaller_interval">
		<failure message="BLOCKED: TypeError: 'int' object is not subscriptable" type="BLOCKED" />
		<system-out>10:24:22 INFO  ==============================================================
10:24:22 INFO  Id:    monitoring_smaller_interval
10:24:22 INFO  Dir:   monitoring/
10:24:22 INFO  Title: Validate the tedge dm agent messages that are produced with smaller sampling interval than the batching window. Here the simulated collectd messages are used
10:24:22 INFO  ==============================================================
10:24:22 INFO  Executed sudo&lt;collectd&gt;, exit status 0
10:24:23 INFO  Executed sudo&lt;collectd_mapper&gt;, exit status 0
10:24:23 INFO  Started sudo&lt;tedge_sub&gt; with process id 25537
10:24:23 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
10:24:24 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
10:24:25 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
10:24:25 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
10:24:26 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
10:24:26 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
10:24:27 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
10:24:28 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
10:24:28 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
10:24:29 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
10:24:29 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
10:24:30 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
10:24:30 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
10:24:31 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
10:24:32 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
10:24:32 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
10:24:33 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
10:24:33 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
10:24:34 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
10:24:34 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
10:24:36 INFO  Executed sudo&lt;kill_out&gt;, exit status 0
10:24:36 INFO  0

10:24:36 WARN  TypeError occurred while running test - 'int' object is not subscriptable
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1296, in __call__
    self.testObj.validate()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/monitoring/monitoring_smaller_interval/run.py", line 104, in validate
    collectd_msg_validation_result=self.validate_json(),
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/monitoring/monitoring_smaller_interval/run.py", line 114, in validate_json
    if not self.validate_time():
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/monitoring/monitoring_smaller_interval/run.py", line 134, in validate_time
    if self.js_msg["time"]:
TypeError: 'int' object is not subscriptable
10:24:36 WARN  TypeError: 'int' object is not subscriptable ... blocked 
10:24:36 INFO  
10:24:36 INFO  cleanup:
10:24:36 INFO  monitoring_cleanup
10:24:37 INFO  Executed sudo&lt;collectd_mapper&gt;, exit status 0
10:24:37 INFO  
10:24:37 INFO  Test duration: 15.00 secs
10:24:37 INFO  Test final outcome:  BLOCKED
10:24:37 INFO  Test outcome reason: TypeError: 'int' object is not subscriptable
10:24:37 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="monitoring_with_collectd" tests="1" errors="0" failures="0" skipped="1" time="0"><testcase classname="MonitoringWithCollectd" name="monitoring_with_collectd">
		<skipped message="SKIPPED: Test execution disabled due to #792 (instable)" />
		<system-out>10:24:37 INFO  ==============================================================
10:24:37 INFO  Id:    monitoring_with_collectd
10:24:37 INFO  Dir:   monitoring/
10:24:37 INFO  Title: Validate tedge dm agent messages that are produced by batching the collectd messages
10:24:37 INFO  ==============================================================
10:24:37 INFO  Test execution disabled due to #792 (instable) ... skipped
10:24:37 INFO  
10:24:37 INFO  Test duration: 0.02 secs
10:24:37 INFO  Test final outcome:  SKIPPED
10:24:37 INFO  Test outcome reason: Test execution disabled due to #792 (instable)
10:24:37 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="monitoring_with_simulated_messages" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="MonitoringWithSimulatedMessages" name="monitoring_with_simulated_messages">
		<failure message="BLOCKED: TypeError: 'int' object is not subscriptable" type="BLOCKED" />
		<system-out>10:24:37 INFO  ==============================================================
10:24:37 INFO  Id:    monitoring_with_simulated_messages
10:24:37 INFO  Dir:   monitoring/
10:24:37 INFO  Title: Validate the tedge dm agent messages that are produced from the simulated collectd messages
10:24:37 INFO  ==============================================================
10:24:37 INFO  Executed sudo&lt;collectd&gt;, exit status 0
10:24:38 INFO  Executed sudo&lt;collectd_mapper&gt;, exit status 0
10:24:38 INFO  Started sudo&lt;tedge_sub&gt; with process id 25592
10:24:39 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
10:24:39 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
10:24:41 INFO  Executed sudo&lt;kill_out&gt;, exit status 0
10:24:41 INFO  0

10:24:41 WARN  TypeError occurred while running test - 'int' object is not subscriptable
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1296, in __call__
    self.testObj.validate()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/monitoring/monitoring_with_simulated_messages/run.py", line 98, in validate
    collectd_msg_validation_result=self.validate_json(),
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/monitoring/monitoring_with_simulated_messages/run.py", line 108, in validate_json
    if not self.validate_time():
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/monitoring/monitoring_with_simulated_messages/run.py", line 127, in validate_time
    if self.js_msg["time"]:
TypeError: 'int' object is not subscriptable
10:24:41 WARN  TypeError: 'int' object is not subscriptable ... blocked 
10:24:41 INFO  
10:24:41 INFO  cleanup:
10:24:41 INFO  monitoring_cleanup
10:24:41 INFO  Executed sudo&lt;collectd_mapper&gt;, exit status 0
10:24:41 INFO  
10:24:41 INFO  Test duration: 4.44 secs
10:24:41 INFO  Test final outcome:  BLOCKED
10:24:41 INFO  Test outcome reason: TypeError: 'int' object is not subscriptable
10:24:41 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="mqtt_port_change_connection_fails" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="MqttPortChangeConnectionFails" name="mqtt_port_change_connection_fails" />
</testsuite><testsuite name="mqtt_port_change_connection_works" tests="1" errors="0" failures="0" skipped="1" time="0"><testcase classname="MqttPortChangeConnectionWorks" name="mqtt_port_change_connection_works">
		<skipped message="SKIPPED: failing intermitently" />
		<system-out>10:23:27 INFO  ==============================================================
10:23:27 INFO  Id:    mqtt_port_change_connection_works
10:23:27 INFO  Dir:   misc_features/
10:23:27 INFO  Title: mqtt port change connection works for all the thin-edge services that use the mosquitto broker
10:23:27 INFO  ==============================================================
10:23:27 INFO  failing intermitently ... skipped
10:23:27 INFO  
10:23:27 INFO  Test duration: 0.03 secs
10:23:27 INFO  Test final outcome:  SKIPPED
10:23:27 INFO  Test outcome reason: failing intermitently
10:23:27 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="mqtt_port_set" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="MqttPortSet" name="mqtt_port_set" />
</testsuite><testsuite name="mqtt_port_unset" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="MqttPortUnSet" name="mqtt_port_unset" />
</testsuite><testsuite name="publish_flux" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PublishFlux" name="publish_flux" />
</testsuite><testsuite name="publish_sawmill" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PublishSawmill" name="publish_sawmill" />
</testsuite><testsuite name="runtime_start_two_mappers" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="RuntimeMultiMappers" name="runtime_start_two_mappers">
		<failure message="FAILED: Grep on tedge_mapper2.err contains &quot;Error: Couldn't acquire file lock.&quot;" type="FAILED" />
		<system-out>10:37:41 INFO  ==============================================================
10:37:41 INFO  Id:    runtime_start_two_mappers
10:37:41 INFO  Dir:   tedge_mapper_c8y/
10:37:41 INFO  Title: Validate two mappers can't be started at the same time
10:37:41 INFO  ==============================================================
10:37:42 INFO  Executed sudo&lt;tedge_mapper1&gt;, exit status 0
10:37:42 INFO  Waiting for 0.1 seconds
10:37:42 INFO  Executed sudo&lt;tedge_mapper2&gt;, exit status 1
10:37:43 INFO  Executed sudo&lt;tedge_mapper1&gt;, exit status 0
10:37:43 WARN  Grep on tedge_mapper2.err contains "Error: Couldn't acquire file lock." ... failed [run.py:46]
10:37:43 INFO  
10:37:43 INFO  Test duration: 1.79 secs
10:37:43 INFO  Test final outcome:  FAILED
10:37:43 INFO  Test outcome reason: Grep on tedge_mapper2.err contains "Error: Couldn't acquire file lock."
10:37:43 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="sm_tedge_mapper_init_session" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeSMMapperInitSession" name="sm_tedge_mapper_init_session" />
</testsuite><testsuite name="smoketest_json" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="SmokeTestJson" name="smoketest_json">
		<failure message="BLOCKED: python3&lt;stdout&gt; returned non-zero exit code 1" type="BLOCKED" />
		<system-out>10:27:22 INFO  ==============================================================
10:27:22 INFO  Id:    smoketest_json
10:27:22 INFO  Dir:   smoketests/
10:27:22 INFO  Title: Smoke Test Json
10:27:22 INFO  ==============================================================
10:27:22 INFO  EnvironmentC8y Setup
10:27:22 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
10:27:23 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 5 seconds
10:27:23 INFO  Delaying execution by 6 seconds
10:27:42 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 13 secs
10:27:43 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
10:27:44 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
10:27:44 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
10:28:01 WARN  Executed python3&lt;stdout&gt;, exit status 1
10:28:01 INFO  Contents of stdout.out: 
  Publishing sawtooth with delay 100ms height 20 iterations 1 template flux will cause 20 publishs.
  Iteraton: 0
  Execution took 2 s 2040 ms
  Publish rate: 102.000 ms/pub
  Publish per second: 9.804 pub/s
  Act: Publishing values with temperature_publisher
  Waiting 3s for values to arrive in C8y
  Assert: Retrieving data via REST interface
  Gathering values from 2022-05-02 10:27:51+00:00 to 2022-05-02 10:28:01+00:00
  Found 0 recorded values:
  Retrieved values:
  Expected:  0.0  ...  19.0
  Data verification FAILED
  -----

10:28:02 WARN  python3&lt;stdout&gt; returned non-zero exit code 1 ... blocked 
10:28:02 WARN  Aborted test due to BLOCKED outcome
10:28:02 INFO  
10:28:02 INFO  cleanup:
10:28:05 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
10:28:06 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
10:28:06 INFO  
10:28:06 INFO  Test duration: 44.21 secs
10:28:06 INFO  Test final outcome:  BLOCKED
10:28:06 INFO  Test outcome reason: python3&lt;stdout&gt; returned non-zero exit code 1
10:28:06 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="smoketest_json_400_samples_10ms" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="SmoketestJson400Samples10ms" name="smoketest_json_400_samples_10ms">
		<failure message="BLOCKED: python3&lt;stdout&gt; returned non-zero exit code 1" type="BLOCKED" />
		<system-out>10:28:06 INFO  ==============================================================
10:28:06 INFO  Id:    smoketest_json_400_samples_10ms
10:28:06 INFO  Dir:   smoketests/
10:28:06 INFO  Title: Smoketest Json 400 Samples 10ms Delay
10:28:06 INFO  ==============================================================
10:28:06 INFO  EnvironmentC8y Setup
10:28:07 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
10:28:07 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 4 seconds
10:28:07 INFO  Delaying execution by 7 seconds
10:28:28 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 13 secs
10:28:28 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
10:28:29 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
10:28:29 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
10:28:58 WARN  Executed python3&lt;stdout&gt;, exit status 1
10:28:58 INFO  Contents of stdout.out: 
  Publishing sawtooth with delay 10ms height 400 iterations 1 template flux will cause 400 publishs.
  Iteraton: 0
  Execution took 4 s 4174 ms
  Publish rate: 10.435 ms/pub
  Publish per second: 95.831 pub/s
  Act: Publishing values with temperature_publisher
  Waiting 3s for values to arrive in C8y
  Assert: Retrieving data via REST interface
  Gathering values from 2022-05-02 10:28:38+00:00 to 2022-05-02 10:28:58+00:00
  Found 0 recorded values:
  Retrieved values:
  Expected:  0.0  ...  399.0
  Data verification FAILED
  -----

10:28:58 WARN  python3&lt;stdout&gt; returned non-zero exit code 1 ... blocked 
10:28:58 WARN  Aborted test due to BLOCKED outcome
10:28:58 INFO  
10:28:58 INFO  cleanup:
10:29:02 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
10:29:03 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
10:29:03 INFO  
10:29:03 INFO  Test duration: 56.94 secs
10:29:03 INFO  Test final outcome:  BLOCKED
10:29:03 INFO  Test outcome reason: python3&lt;stdout&gt; returned non-zero exit code 1
10:29:03 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="smoketest_json_400_samples_20ms" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="SmoketestJson400Samples20ms" name="smoketest_json_400_samples_20ms">
		<failure message="BLOCKED: python3&lt;stdout&gt; returned non-zero exit code 1" type="BLOCKED" />
		<system-out>10:29:03 INFO  ==============================================================
10:29:03 INFO  Id:    smoketest_json_400_samples_20ms
10:29:03 INFO  Dir:   smoketests/
10:29:03 INFO  Title: Smoketest Json 400 Samples 20ms Delay
10:29:03 INFO  ==============================================================
10:29:03 INFO  EnvironmentC8y Setup
10:29:04 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
10:29:04 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 4 seconds
10:29:04 INFO  Delaying execution by 7 seconds
10:29:25 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 13 secs
10:29:25 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
10:29:26 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
10:29:26 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
10:29:56 WARN  Executed python3&lt;stdout&gt;, exit status 1
10:29:56 INFO  Contents of stdout.out: 
  Publishing sawtooth with delay 10ms height 400 iterations 1 template flux will cause 400 publishs.
  Iteraton: 0
  Execution took 4 s 4231 ms
  Publish rate: 10.578 ms/pub
  Publish per second: 94.540 pub/s
  Act: Publishing values with temperature_publisher
  Waiting 3s for values to arrive in C8y
  Assert: Retrieving data via REST interface
  Gathering values from 2022-05-02 10:29:35+00:00 to 2022-05-02 10:29:55+00:00
  Found 0 recorded values:
  Retrieved values:
  Expected:  0.0  ...  399.0
  Data verification FAILED
  -----

10:29:56 WARN  python3&lt;stdout&gt; returned non-zero exit code 1 ... blocked 
10:29:56 WARN  Aborted test due to BLOCKED outcome
10:29:56 INFO  
10:29:56 INFO  cleanup:
10:30:00 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
10:30:00 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
10:30:00 INFO  
10:30:00 INFO  Test duration: 56.98 secs
10:30:00 INFO  Test final outcome:  BLOCKED
10:30:00 INFO  Test outcome reason: python3&lt;stdout&gt; returned non-zero exit code 1
10:30:00 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="smoketest_smartrest" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="SmoketestSmartRest" name="smoketest_smartrest">
		<failure message="BLOCKED: python3&lt;stdout&gt; returned non-zero exit code 1" type="BLOCKED" />
		<system-out>10:30:00 INFO  ==============================================================
10:30:00 INFO  Id:    smoketest_smartrest
10:30:00 INFO  Dir:   smoketests/
10:30:00 INFO  Title: Smoketest SmartREST
10:30:00 INFO  ==============================================================
10:30:00 INFO  EnvironmentC8y Setup
10:30:01 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
10:30:01 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 4 seconds
10:30:01 INFO  Delaying execution by 7 seconds
10:30:22 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
10:30:23 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
10:30:23 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
10:30:24 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
10:30:44 WARN  Executed python3&lt;stdout&gt;, exit status 1
10:30:44 INFO  Contents of stdout.out: 
  Act: Publishing values with tedge pub
  Waiting 3s for values to arrive in C8y
  Assert: Retrieving data via REST interface
  Gathering values from 2022-05-02 10:30:34+00:00 to 2022-05-02 10:30:44+00:00
  Found 0 recorded values:
  Retrieved values:
  Expected:  0.0  ...  19.0
  Data verification FAILED
  -----

10:30:44 WARN  python3&lt;stdout&gt; returned non-zero exit code 1 ... blocked 
10:30:44 WARN  Aborted test due to BLOCKED outcome
10:30:44 INFO  
10:30:44 INFO  cleanup:
10:30:48 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
10:30:49 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
10:30:49 INFO  
10:30:49 INFO  Test duration: 48.59 secs
10:30:49 INFO  Test final outcome:  BLOCKED
10:30:49 INFO  Test outcome reason: python3&lt;stdout&gt; returned non-zero exit code 1
10:30:49 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="smoketest_smartrest_400_samples_1Xms" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="SmoketestSmartRest400Samples1Xms" name="smoketest_smartrest_400_samples_1Xms">
		<failure message="BLOCKED: python3&lt;stdout&gt; returned non-zero exit code 1" type="BLOCKED" />
		<system-out>10:30:49 INFO  ==============================================================
10:30:49 INFO  Id:    smoketest_smartrest_400_samples_1Xms
10:30:49 INFO  Dir:   smoketests/
10:30:49 INFO  Title: Smoketest Smart Rest 400 Samples 1Xms Delay
10:30:49 INFO  ==============================================================
10:30:49 INFO  EnvironmentC8y Setup
10:30:49 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
10:30:50 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 5 seconds
10:30:50 INFO  Delaying execution by 6 seconds
10:31:09 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 13 secs
10:31:10 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
10:31:10 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
10:31:11 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
10:32:32 WARN  Executed python3&lt;stdout&gt;, exit status 1, duration 40 secs
10:32:32 INFO  Contents of stdout.out: 
  Act: Publishing values with tedge pub
  Waiting 3s for values to arrive in C8y
  Assert: Retrieving data via REST interface
  Gathering values from 2022-05-02 10:31:51+00:00 to 2022-05-02 10:32:31+00:00
  Found 0 recorded values:
  Retrieved values:
  Expected:  0.0  ...  199.0
  Data verification FAILED
  -----

10:32:32 WARN  python3&lt;stdout&gt; returned non-zero exit code 1 ... blocked 
10:32:32 WARN  Aborted test due to BLOCKED outcome
10:32:32 INFO  
10:32:32 INFO  cleanup:
10:32:36 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
10:32:36 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
10:32:36 INFO  
10:32:36 INFO  Test duration: 107.34 secs
10:32:36 INFO  Test final outcome:  BLOCKED
10:32:36 INFO  Test outcome reason: python3&lt;stdout&gt; returned non-zero exit code 1
10:32:36 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="smoketest_smartrest_400_samples_20ms" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="SmoketestSmartRest400Samples20ms" name="smoketest_smartrest_400_samples_20ms">
		<failure message="BLOCKED: python3&lt;stdout&gt; returned non-zero exit code 1" type="BLOCKED" />
		<system-out>10:32:36 INFO  ==============================================================
10:32:36 INFO  Id:    smoketest_smartrest_400_samples_20ms
10:32:36 INFO  Dir:   smoketests/
10:32:36 INFO  Title: Smoketest SmartRest 400 Samples 20ms Delay
10:32:36 INFO  ==============================================================
10:32:36 INFO  EnvironmentC8y Setup
10:32:37 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
10:32:37 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 4 seconds
10:32:37 INFO  Delaying execution by 7 seconds
10:32:58 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 13 secs
10:32:59 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
10:32:59 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
10:33:00 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
10:34:18 WARN  Executed python3&lt;stdout&gt;, exit status 1, duration 38 secs
10:34:18 INFO  Contents of stdout.out: 
  Act: Publishing values with tedge pub
  Waiting 3s for values to arrive in C8y
  Assert: Retrieving data via REST interface
  Gathering values from 2022-05-02 10:33:38+00:00 to 2022-05-02 10:34:18+00:00
  Found 0 recorded values:
  Retrieved values:
  Expected:  0.0  ...  199.0
  Data verification FAILED
  -----

10:34:18 WARN  python3&lt;stdout&gt; returned non-zero exit code 1 ... blocked 
10:34:18 WARN  Aborted test due to BLOCKED outcome
10:34:18 INFO  
10:34:18 INFO  cleanup:
10:34:22 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
10:34:23 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
10:34:23 INFO  
10:34:23 INFO  Test duration: 106.31 secs
10:34:23 INFO  Test final outcome:  BLOCKED
10:34:23 INFO  Test outcome reason: python3&lt;stdout&gt; returned non-zero exit code 1
10:34:23 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="tedge_agent_user_sudo_access" tests="1" errors="0" failures="0" skipped="1" time="0"><testcase classname="TedgeAgentUserSudoAccess" name="tedge_agent_user_sudo_access">
		<skipped message="SKIPPED: #955 Observation: Exit code of the apt plugin when called with no arguments" />
		<system-out>10:34:50 INFO  ==============================================================
10:34:50 INFO  Id:    tedge_agent_user_sudo_access
10:34:50 INFO  Dir:   tedge/
10:34:50 INFO  Title: Validate tedge-agent user has a limited sudo right
10:34:50 INFO  ==============================================================
10:34:50 INFO  #955 Observation: Exit code of the apt plugin when called with no arguments ... skipped
10:34:50 INFO  
10:34:50 INFO  Test duration: 0.02 secs
10:34:50 INFO  Test final outcome:  SKIPPED
10:34:50 INFO  Test outcome reason: #955 Observation: Exit code of the apt plugin when called with no arguments
10:34:50 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="tedge_connect_test_negative" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeConnectTestNegative" name="tedge_connect_test_negative" />
</testsuite><testsuite name="tedge_connect_test_positive" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeConnectTestPositive" name="tedge_connect_test_positive" />
</testsuite><testsuite name="tedge_connect_test_sm_services" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeConnectTestSMServices" name="tedge_connect_test_sm_services" />
</testsuite><testsuite name="tedge_disconnect_test_sm_services" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeDisConnectTestSMServices" name="tedge_disconnect_test_sm_services" />
</testsuite><testsuite name="tedge_mapper_az_negative" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeMapperAzError" name="tedge_mapper_az_negative" />
</testsuite><testsuite name="tedge_mapper_az_with_timestamp" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeMapperAzWithTimestamp" name="tedge_mapper_az_with_timestamp" />
</testsuite><testsuite name="tedge_mapper_az_without_timestamp" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeMapperAzWithoutTimestamp" name="tedge_mapper_az_without_timestamp" />
</testsuite><testsuite name="tedge_mapper_c8y_alarms" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="TedgeMapperC8yAlarm" name="tedge_mapper_c8y_alarms">
		<failure message="BLOCKED: IndexError: list index out of range" type="BLOCKED" />
		<system-out>10:38:20 INFO  ==============================================================
10:38:20 INFO  Id:    tedge_mapper_c8y_alarms
10:38:20 INFO  Dir:   tedge_mapper_c8y/
10:38:20 INFO  Title: Validate that alarms sent to tedge/alarms topic \ gets mapped as alarms in Cumulocity
10:38:20 INFO  ==============================================================
10:38:20 INFO  EnvironmentC8y Setup
10:38:21 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
10:38:35 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
10:38:36 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
10:38:38 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
10:38:38 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
10:38:39 INFO  Executed tedge, exit status 0
10:38:39 INFO  Executed tedge, exit status 0
10:38:40 INFO  Executed tedge, exit status 0
10:38:40 INFO  Executed tedge, exit status 0
10:38:41 INFO  Executed tedge, exit status 0
10:38:42 WARN  IndexError occurred while running test - list index out of range
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1296, in __call__
    self.testObj.validate()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/tedge_mapper_c8y/tedge_mapper_c8y_alarms/run.py", line 96, in validate
    alarms[0],
IndexError: list index out of range
10:38:42 WARN  IndexError: list index out of range ... blocked 
10:38:42 INFO  
10:38:42 INFO  cleanup:
10:38:42 INFO  Executed tedge, exit status 0
10:38:43 INFO  Executed tedge, exit status 0
10:38:43 INFO  Executed tedge, exit status 0
10:38:47 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
10:38:48 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
10:38:48 INFO  
10:38:48 INFO  Test duration: 27.47 secs
10:38:48 INFO  Test final outcome:  BLOCKED
10:38:48 INFO  Test outcome reason: IndexError: list index out of range
10:38:48 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="tedge_mapper_c8y_child_device_measurement" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="TedgeMapperC8yChildDevice" name="tedge_mapper_c8y_child_device_measurement">
		<failure message="BLOCKED: IndexError: list index out of range" type="BLOCKED" />
		<system-out>10:38:48 INFO  ==============================================================
10:38:48 INFO  Id:    tedge_mapper_c8y_child_device_measurement
10:38:48 INFO  Dir:   tedge_mapper_c8y/
10:38:48 INFO  Title: Validate that measurements sent to tedge/measurements/child topic gets mapped as the child device's measurement in Cumulocity
10:38:48 INFO  ==============================================================
10:38:48 INFO  EnvironmentC8y Setup
10:38:48 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
10:38:49 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 5 seconds
10:38:49 INFO  Delaying execution by 6 seconds
10:39:08 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 13 secs
10:39:09 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
10:39:09 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
10:39:10 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
10:39:11 INFO  Assert that (actual == expected) with actual=None, expected=None ... passed
10:39:11 INFO  Executed tedge_pub, exit status 0
10:39:12 INFO  Executed tedge_pub, exit status 0
10:39:12 INFO  Executed tedge_pub, exit status 0
10:39:12 INFO  Executed tedge_pub, exit status 0
10:39:13 INFO  Executed tedge_pub, exit status 0
10:39:15 INFO  {
    "self": "https://t493319102.eu-latest.cumulocity.com/measurement/measurements/18136621",
    "time": "2022-05-02T10:39:13.271Z",
    "id": "18136621",
    "source": {
        "self": "https://t493319102.eu-latest.cumulocity.com/inventory/managedObjects/18136615",
        "id": "18136615"
    },
    "type": "ThinEdgeMeasurement",
    "temperature": {
        "temperature": {
            "value": 10.0
        }
    }
}
10:39:15 INFO  Assert that (actual == expected) with actual="ThinEdgeMeasurement", expected="ThinEdgeMeasurement" ... passed
10:39:15 INFO  Assert that (actual == expected) with actual=10.0, expected=10 ... passed
10:39:15 INFO  {
    "self": "https://t493319102.eu-latest.cumulocity.com/measurement/measurements/18136617",
    "time": "2022-05-02T10:39:12.007Z",
    "id": "18136617",
    "source": {
        "self": "https://t493319102.eu-latest.cumulocity.com/inventory/managedObjects/18136615",
        "id": "18136615"
    },
    "type": "ThinEdgeMeasurement",
    "temperature": {
        "temperature": {
            "value": 11.0
        }
    }
}
10:39:15 INFO  Assert that (actual == expected) with actual="ThinEdgeMeasurement", expected="ThinEdgeMeasurement" ... passed
10:39:15 INFO  Assert that (actual == expected) with actual=11.0, expected=11 ... passed
10:39:15 INFO  {
    "self": "https://t493319102.eu-latest.cumulocity.com/measurement/measurements/18136616",
    "time": "2021-01-01T10:10:10.100+02:00",
    "id": "18136616",
    "source": {
        "self": "https://t493319102.eu-latest.cumulocity.com/inventory/managedObjects/18136615",
        "id": "18136615"
    },
    "type": "ThinEdgeMeasurement",
    "temperature": {
        "temperature": {
            "value": 12.0
        }
    }
}
10:39:15 INFO  Assert that (actual == expected) with actual="ThinEdgeMeasurement", expected="ThinEdgeMeasurement" ... passed
10:39:15 INFO  Assert that (actual == expected) with actual=12.0, expected=12 ... passed
10:39:15 INFO  Assert that (actual == expected) with actual="2021-01-01T10:10:10.100+02:00", expected="2021-01-01T10:10:10.100+02:00" ... passed
10:39:16 INFO  {
    "self": "https://t493319102.eu-latest.cumulocity.com/measurement/measurements/18136620",
    "time": "2022-05-02T10:39:12.822Z",
    "id": "18136620",
    "source": {
        "self": "https://t493319102.eu-latest.cumulocity.com/inventory/managedObjects/18136619",
        "id": "18136619"
    },
    "type": "ThinEdgeMeasurement",
    "temperature": {
        "temperature": {
            "value": 6.0
        }
    }
}
10:39:16 INFO  Assert that (actual == expected) with actual="ThinEdgeMeasurement", expected="ThinEdgeMeasurement" ... passed
10:39:16 INFO  Assert that (actual == expected) with actual=6.0, expected=6 ... passed
10:39:16 WARN  IndexError occurred while running test - list index out of range
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1296, in __call__
    self.testObj.validate()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/tedge_mapper_c8y/tedge_mapper_c8y_child_device_measurement/run.py", line 133, in validate
    self.project.deviceid
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/environments/cumulocity.py", line 163, in get_last_measurements_from_device
    )[0]
IndexError: list index out of range
10:39:16 WARN  IndexError: list index out of range ... blocked 
10:39:16 INFO  
10:39:16 INFO  cleanup:
10:39:20 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
10:39:21 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
10:39:21 INFO  
10:39:21 INFO  Test duration: 33.16 secs
10:39:21 INFO  Test final outcome:  BLOCKED
10:39:21 INFO  Test outcome reason: IndexError: list index out of range
10:39:21 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="tedge_mapper_c8y_packet_threshold_size" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="TedgeMapperC8yThresholdPacketSize" name="tedge_mapper_c8y_packet_threshold_size">
		<failure message="FAILED: Grep on tedge_sub.out contains &quot;The size of the message received on tedge/measurements is 20489 which is greater than the threshold size of 16184.&quot;" type="FAILED" />
		<system-out>10:39:21 INFO  ==============================================================
10:39:21 INFO  Id:    tedge_mapper_c8y_packet_threshold_size
10:39:21 INFO  Dir:   tedge_mapper_c8y/
10:39:21 INFO  Title: Validate the Cumulocity Mapper accepted threshold packet size when it receives a message on tedge/measurements
10:39:21 INFO  ==============================================================
10:39:22 INFO  Executed sudo&lt;tedge_mapper_c8y&gt;, exit status 0
10:39:22 INFO  Started sudo&lt;tedge_sub&gt; with process id 29998
10:39:22 INFO  Executed sudo&lt;tedge_msg&gt;, exit status 0
10:39:23 INFO  Executed sudo&lt;tedge_pub&gt;, exit status 0
10:39:24 INFO  Executed sudo&lt;kill_out&gt;, exit status 0
10:39:24 WARN  Grep on tedge_sub.out contains "The size of the message received on tedge/measurements is 20489 which is greater than the threshold size of 16184." ... failed [run.py:88]
10:39:24 INFO  
10:39:24 INFO  cleanup:
10:39:24 INFO  mapper_cleanup
10:39:24 INFO  Executed sudo&lt;tedge_mapper_c8y&gt;, exit status 0
10:39:24 INFO  
10:39:24 INFO  Test duration: 3.39 secs
10:39:24 INFO  Test final outcome:  FAILED
10:39:24 INFO  Test outcome reason: Grep on tedge_sub.out contains "The size of the message received on tedge/measurements is 20489 which is greater than the threshold size of 16184."
10:39:24 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="tedge_software_update" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="tedge_software_update" />
</testsuite><testsuite name="valid_device_id" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="ValidateValidDeviceId" name="valid_device_id" />
</testsuite></testsuites>